Translate Zombie Girl 10053.txt

ArdynXT opened the pull request:
Hello all, Wanted to help out with translation. I noticed this fight hadn't been worked on yet so, I tried my hand at translating Zombie Girl to test the waters.  I can guarantee there will be some jank but, when trying out the fight for myself the dialogue started to a fit a lot better after the initial revision.

ArdynXT committed:
Update 10053.txt

ArdynXT committed:
Minor fixes

Forgot to proofread some of the earlier sections

ArdynXT committed:
Revision2

Went through the text after thinking about how to make the dialogue flow better Found a ton of overlooked issues. general grammar, spelling, and word usage

ArdynXT committed:
Revision3

Minor grammar and context issues

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
-くすくす・・・❤仲間になってもやることは変わらないね・・・。\nほら、おっぱいでしてあげる・・・❤\nあとで少しだけ、肩をかじらせてね・・・❤,
-その後、{主人公}がどこに行ったかは誰も知らない。無敗のルーキーの物語は、こうしてその幕を閉じたのだった。,
+ゾンビのモンスター娘。体はひんやりと冷たいが、胸やふとももは柔らかい。相手にゾンビウイルスを感染させるために口を使った誘惑をしてくるが、そうした本能とは別に、性的なことを通して「命」を強く感じられるため元々エッチなことが好き。,A zombie monster girl. Her body is cold to the touch; but her breasts and thighs are soft. She uses her mouth to seduce others and infect them with the zombie virus; but beyond that instinct; she genuinely enjoys sexual acts because they allow her to feel “life” intensely. Necro gave  ...
+口についている赤いものは血ではなくザクロの果汁である。肉を生で食べるのは不衛生だと言ってネクロが赤い果物を代わりに与えたところ、本人は気に入ってよくザクロをかじっている。,\n The red substance on her mouth not blood but pomegranate juice. Necro gave her the red fruit instead; saying eating raw meat was unsanitary; and she took to it; often munching on pomegranates.
```
We'll need to work on lines 4 and 5! 
It isn't coming thru properly in the encyclopedia - I tried some quick fixes but they didn't help. This is likely to be a complicated one, so I'll just note it now to fix later.

Also will need to localize it, since it reads a bit rough at the moment.

ArdynXT replied:
Took some liberties with the entry it should flow a lot better now.

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ゾンビのモンスター娘。体はひんやりと冷たいが、胸やふとももは柔らかい。相手にゾンビウイルスを感染させるために口を使った誘惑をしてくるが、そうした本能とは別に、性的なことを通して「命」を強く感じられるため元々エッチなことが好き。,A zombie monster girl. Her body is cold to the touch; but her breasts and thighs are soft. She uses her mouth to seduce others and infect them with the zombie virus; but beyond that instinct; she genuinely enjoys sexual acts because they allow her to feel “life” intensely. Necro gave  ...
+口についている赤いものは血ではなくザクロの果汁である。肉を生で食べるのは不衛生だと言ってネクロが赤い果物を代わりに与えたところ、本人は気に入ってよくザクロをかじっている。,\n The red substance on her mouth not blood but pomegranate juice. Necro gave her the red fruit instead; saying eating raw meat was unsanitary; and she took to it; often munching on pomegranates.
+[1].このカードは「ゾンビ」状態を持ち。戦闘時に相手に「ゾンビ」状態を付与する。\n[2].このカードが墓地から特殊召喚されたとき発動する。フィールド上の「ゾンビ」状態のモンスターの数×100ポイント、自身の攻撃力をアップさせる。,"[1]. This card possesses the “Zombie” status. During combat; it inflicts the “Zombie” status on the opponent. \n[2]. When this card is Special Summoned from the Graveyard; activate this effect. Increase its ATK by 100 points for each “Zombie” monster on the field.
+[3].このカードが戦闘で「ゾンビ」状態の相手モンスターを破壊したとき発動する。破壊したモンスターのパラメーターを半分にして自分フィールド上に特殊召喚する。,"[3]. When this card destroys an opponent's monster in the “Zombie” state during battle; activate this effect. Special Summon that monster to your field with half its original parameters."
```
Lines 6 and 7 have extra quote marks - remove leading quote from 6 and surrounding quotes on 7

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+[1].このカードは「ゾンビ」状態を持ち。戦闘時に相手に「ゾンビ」状態を付与する。\n[2].このカードが墓地から特殊召喚されたとき発動する。フィールド上の「ゾンビ」状態のモンスターの数×100ポイント、自身の攻撃力をアップさせる。,"[1]. This card possesses the “Zombie” status. During combat; it inflicts the “Zombie” status on the opponent. \n[2]. When this card is Special Summoned from the Graveyard; activate this effect. Increase its ATK by 100 points for each “Zombie” monster on the field.
+[3].このカードが戦闘で「ゾンビ」状態の相手モンスターを破壊したとき発動する。破壊したモンスターのパラメーターを半分にして自分フィールド上に特殊召喚する。,"[3]. When this card destroys an opponent's monster in the “Zombie” state during battle; activate this effect. Special Summon that monster to your field with half its original parameters."
+あなたの体、おいしそう・・・❤,Your body looks delicious...❤
+かじっていい？\nかじっている間は、代わりに手コキしてあげるから・・・❤\nしこ、しこ・・・❤,Can I dine on you?❤ I’ll touch your cock like this…❤*Sshiko; shiko*...❤
```
"shiko shiko" is an onomatopoeia for stroking, kind of similar to saying "stroke, stroke" out loud.
 
Usually, we'd localize it a bit - something like "Stroking up and down..." could work, I think.

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+[3].このカードが戦闘で「ゾンビ」状態の相手モンスターを破壊したとき発動する。破壊したモンスターのパラメーターを半分にして自分フィールド上に特殊召喚する。,"[3]. When this card destroys an opponent's monster in the “Zombie” state during battle; activate this effect. Special Summon that monster to your field with half its original parameters."
+あなたの体、おいしそう・・・❤,Your body looks delicious...❤
+かじっていい？\nかじっている間は、代わりに手コキしてあげるから・・・❤\nしこ、しこ・・・❤,Can I dine on you?❤ I’ll touch your cock like this…❤*Sshiko; shiko*...❤
+あ、胸、見てる・・・❤\nけっこう大きいから、つい目で追っちゃうのかな・・・？\n男ってわかりやすい・・・❤,Oh~❤  Focused on the swaying of my breasts?❤ Hoping my nipples will slip out from behind all this hair?❤ Men are so easy to understand... ❤
```
This one takes some liberties in the original meaning, but I feel like it fits the intent. 
(Instead of nipples slipping out, I think the original line was closer to "They're quite large; so I suppose your eyes can't help but follow them")

If the change is OK'd by @Giratena, I'd still recommend a small adjustment, to make the first bit a statement rather than having two questions in a row.
 > **Oh; so you're** focused on the swaying of my **breasts...❤**

ArdynXT replied:
Got rid of the double question. I will wait on resolving though just to see what @Giratena says.

Giratena replied:
Getting rid of the double question is good! I will resolve this ^^

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+かじっていい？\nかじっている間は、代わりに手コキしてあげるから・・・❤\nしこ、しこ・・・❤,Can I dine on you?❤ I’ll touch your cock like this…❤*Sshiko; shiko*...❤
+あ、胸、見てる・・・❤\nけっこう大きいから、つい目で追っちゃうのかな・・・？\n男ってわかりやすい・・・❤,Oh~❤  Focused on the swaying of my breasts?❤ Hoping my nipples will slip out from behind all this hair?❤ Men are so easy to understand... ❤
+ねえ、おっぱい好きなら、挟んで、気持ちよくしてあげるよ・・・❤\nでも射精したらその瞬間、あなたのお肉食べさせてね・・・。ふふふ❤,Hey do you like my tits? I can push them together around your cock...❤❤ If I satisfy your urges;  can I satisfy my appetite afterwards? Hmmm...❤
+あは、舌が気になるの・・・？\nさっきから舌なめずりしてたから期待させちゃったのかな・・・❤,Haha; Are you afraid of putting it in my mouth?❤ Don’t worry I’ll be gentle...❤
```
"tongue" (舌) is specifically being mentioned in the Japanese. 

I think the line is supposed to be 
> Aha; are you curious about my tongue...? I wonder if I got your hopes up since I've been licking my lips so much...?❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あは、舌が気になるの・・・？\nさっきから舌なめずりしてたから期待させちゃったのかな・・・❤,Haha; Are you afraid of putting it in my mouth?❤ Don’t worry I’ll be gentle...❤
+ほんとに食べたいのはあなたのお肉だけれど、\nいいよ。この口でフェラしてあげる❤\nお腹空いてるから、エナジーごちそうしてね・・・❤れろーっ❤,I really want to take a bite of you; but I'll just have to settle for your mana...❤ I am very hungry; so please treat me...❤ *Rerroooooooo*❤
+スカートの中、見たい？\nふふ、こんなときでもパンツが気になるなんて変態だね・・・❤,Trying to get a peak under my skirt?❤ You are such a pervert; even with your life on the line you can't help yourself...❤
+ミニスカートがひらひらして、気が散っちゃったの・・・？\nほら、ぴらり❤\n見て釘付けになっているところ、噛みついてあげる・・・❤,Did you catch a glimpse of my panties...? ❤ I'll just take a bite while you are preoccupied…❤
```
Some words were dropped from the original, I think it was something like:

> Aw is the fluttering of my miniskirt distracting you...? Look; a little peek.❤ And while your eyes are glued to that; I'll just take a bite...❤

I do prefer your "I'll just take a bite while you are preoccupied…❤" localization, but I don't want to drop the
"ほら、ぴらり❤" (Hora, *flutter sfx*).

(Note: We don't have to go with "Look; a little peek.❤" as the translation for that bit, but I think having her intentionally fluttering her skirt changes the situation)

ArdynXT replied:
I have taken a look and made some changes let me know if they fit better!

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あううっ・・・❤きもちいい・・・❤,Ohhhhhh...❤ So good...❤
+くすくす・・・❤\n朦朧としながらシコシコするの気持ちいい・・・？,*Giggle*... ❤ Does it feel good to get pleasured while your head is spinning?❤
+命が危なくなると、子孫を残すために、\nペニスが固くなるっていうけど、\nほんとみたい・・・❤,They say that when your life is in danger; your dick feels extra pleasure as a last resort to preserve your genes; Can you feel it?❤
+噛まれる前よりガチガチに固くなって、\nシコシコするたびに顔がどんどん蕩けて、\nふふ・・・❤子孫を残すために、びゅってしたくてたまらないんだね❤,You are getting so hard;I think it’s even harder than before you were bitten; and every time I bite; That naughty expression on your face becomes even harder to hide. \nyour cock is so eager to be sucked hoping to pass on it’s seed...❤❤
```
General note: avoid using "\n" in the English translations except for the Encyclopedia entry and in the abilities.

It just creates a new line, so it won't destroy the game, but we're relying on the text wrapping instead.

See lines 70 and 118 as well.

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ほら、わたしも{自分}のお手伝い。\n顔におっぱいを乗せてあげる❤,Here; I'll help out by putting my boobs on your face❤
+盗賊の用心棒のおっぱいなんかに負けちゃダメよ。\nコノハちゃんの弟子くん❤\nえい❤,Look at you losing yourself in the embrace of a bandits chest.
+あう、あうううっ・・・❤❤❤,Oh; ahhhhh...❤❤❤❤
+ふふ❤わたしのおっぱいやわらかいでしょう？\n{自分}のと比べるとあったかいから、\n余計にふわふわに感じちゃうかも❤,My boobs are soft; aren't they? They are much warmer than my zombie girl; so they might feel even fluffier.
```
As a note, the bracketed bits should generally remain untranslated, as they call names from the code.
In this case, I believe {自分} is usually replaced by the card's name (Zombie Girl), so you hard-translated it appropriately, but it is something to keep an eye on!

> My boobs are soft; aren't they? They are much warmer than my **{自分}**; so they might feel even fluffier.


Bracketed lines to review in this file are 93, 96, 98, 132, and 138

{自分} - card name
{主人公} - becomes the name you entered for your player

ArdynXT replied:
Thank you! I was not sure how this worked!!!!!

WisDZ reviewed (commented):
Sorry it has taken so long to get a review started on this one!

I've only added a few initial notes so far. The full review is not done yet, but I figured I'd post this now rather than make you wait even longer for it to be fully done.

The first six comments are notes on specific lines, but the last two comments are generalized rules to follow.

Feel free to reply to the comments, or if you prefer, you can reach out to me in Discord! When a comment is resolved, click "Resolve conversation" to clear it.

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ミニスカートがひらひらして、気が散っちゃったの・・・？\nほら、ぴらり❤\n見て釘付けになっているところ、噛みついてあげる・・・❤,Did you catch a glimpse of my panties...? ❤ I'll just take a bite while you are preoccupied…❤
+ふふ・・・❤\nゾンビのわたしにおねだりするの？\nあなた、いい趣味してるね・・・❤,You're going to beg me; a zombie? You have good taste...❤
+胸に、口に、手に・・・。\nちゃんと体は動くから、いろいろつかって気持ちよくしてあげる・・・❤\nでも、ふふ、代わりにあなたのことかじっちゃうかも・・・❤,On my chest?❤ in my mouth?❤ Maybe in my hand....❤ I can use all sorts of things to make you feel good...❤ but I might just bite you instead...❤
+んっ・・・❤口の中に濃いのが注がれてくる・・・❤\n生気にあふれてて、おいしい・・・❤,Mmmm...❤ Mmmm…❤ Lush and delicious...❤
```
The middle sentence needs to be included!
I'm recommending "lush" -> "full of life" to be consistent with the card description, though it is probably fine as-is

> Mmmm...❤ **Something thick is being poured into my mouth...❤ So full of life** and delicious...❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ふふ・・・❤\nゾンビのわたしにおねだりするの？\nあなた、いい趣味してるね・・・❤,You're going to beg me; a zombie? You have good taste...❤
+胸に、口に、手に・・・。\nちゃんと体は動くから、いろいろつかって気持ちよくしてあげる・・・❤\nでも、ふふ、代わりにあなたのことかじっちゃうかも・・・❤,On my chest?❤ in my mouth?❤ Maybe in my hand....❤ I can use all sorts of things to make you feel good...❤ but I might just bite you instead...❤
+んっ・・・❤口の中に濃いのが注がれてくる・・・❤\n生気にあふれてて、おいしい・・・❤,Mmmm...❤ Mmmm…❤ Lush and delicious...❤
+射精して放心してる・・・？お口の中で完全にペニスが無防備。\nかみきっちゃってもいいけど、\nんふふ、エナジーがおいしいから許してあげるね・・・❤,Ejaculating and letting loose...?❤ Your defenseless cock in the mouth of a zombie. I could chew it all up; but that would be such a waste; your mana tastes so good... ❤
```
Recommending some changes to make the ideas flow in English:

> **Are you in a daze after ejaculating so much...? That leaves** your cock defenseless in the mouth of a zombie. I could **bite it off; but... Fufu; I'll spare you because** your mana tastes so good...❤

ArdynXT replied:
Made some changes here!

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+胸に、口に、手に・・・。\nちゃんと体は動くから、いろいろつかって気持ちよくしてあげる・・・❤\nでも、ふふ、代わりにあなたのことかじっちゃうかも・・・❤,On my chest?❤ in my mouth?❤ Maybe in my hand....❤ I can use all sorts of things to make you feel good...❤ but I might just bite you instead...❤
+んっ・・・❤口の中に濃いのが注がれてくる・・・❤\n生気にあふれてて、おいしい・・・❤,Mmmm...❤ Mmmm…❤ Lush and delicious...❤
+射精して放心してる・・・？お口の中で完全にペニスが無防備。\nかみきっちゃってもいいけど、\nんふふ、エナジーがおいしいから許してあげるね・・・❤,Ejaculating and letting loose...?❤ Your defenseless cock in the mouth of a zombie. I could chew it all up; but that would be such a waste; your mana tastes so good... ❤
+胸に挟まれて、きもちよくびゅーびゅー・・・・❤\nそれじゃ、約束通り、あなたのお腹にかじりついちゃうね・・・❤,I'm going to bite you as promised... ❤
```
This one is missing a lot of context!

The full line is more like "Squeezed between my breasts; you're spurting so pleasurably...❤ Now then; as promised; I'm going to bite into your belly...❤"

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+んっ・・・❤口の中に濃いのが注がれてくる・・・❤\n生気にあふれてて、おいしい・・・❤,Mmmm...❤ Mmmm…❤ Lush and delicious...❤
+射精して放心してる・・・？お口の中で完全にペニスが無防備。\nかみきっちゃってもいいけど、\nんふふ、エナジーがおいしいから許してあげるね・・・❤,Ejaculating and letting loose...?❤ Your defenseless cock in the mouth of a zombie. I could chew it all up; but that would be such a waste; your mana tastes so good... ❤
+胸に挟まれて、きもちよくびゅーびゅー・・・・❤\nそれじゃ、約束通り、あなたのお腹にかじりついちゃうね・・・❤,I'm going to bite you as promised... ❤
+だいじょうぶ、肉はかじりとらないで甘噛みするだけ・・・❤\nちょっとだけ歯型をつけて、それで許してあげる・・・❤\nがぶがぶ・・・♪,Don't worry; I won't bite too hard; I'll just chew on it...❤ I'll give you a little nibble and let you go... ❤Gulp!
```
Recommending changes for clarity, what do you think about the below?

> Don't worry; I won't bite too hard; just a little nibble...❤ Enough to leave a mark; and then I'll let you go...❤ Chomp chomp...♪

ArdynXT replied:
Much better! I was confused by the original hard translations from google

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+射精して放心してる・・・？お口の中で完全にペニスが無防備。\nかみきっちゃってもいいけど、\nんふふ、エナジーがおいしいから許してあげるね・・・❤,Ejaculating and letting loose...?❤ Your defenseless cock in the mouth of a zombie. I could chew it all up; but that would be such a waste; your mana tastes so good... ❤
+胸に挟まれて、きもちよくびゅーびゅー・・・・❤\nそれじゃ、約束通り、あなたのお腹にかじりついちゃうね・・・❤,I'm going to bite you as promised... ❤
+だいじょうぶ、肉はかじりとらないで甘噛みするだけ・・・❤\nちょっとだけ歯型をつけて、それで許してあげる・・・❤\nがぶがぶ・・・♪,Don't worry; I won't bite too hard; I'll just chew on it...❤ I'll give you a little nibble and let you go... ❤Gulp!
+あは、パンツに見惚れて、\nかみつかれるのと同時に射精しちゃった・・・❤,Haha; You were so entranced by my skirt that you came right as I bit you...❤
```
パンツ (pantsu) 
so
skirt -> panties

WisDZ reviewed (commented):
Review part 2!

I meant to get more done today, but I got distracted.

ArdynXT committed:
Addressed issues

Took a look through the conflicts and reworked or copy pasted the adjustments that were needed.

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+胸に挟まれて、きもちよくびゅーびゅー・・・・❤\nそれじゃ、約束通り、あなたのお腹にかじりついちゃうね・・・❤,Squeezed between my breasts; you're spurting so pleasurably...❤ Now then; as promised; I'm going to bite into your belly...❤
+だいじょうぶ、肉はかじりとらないで甘噛みするだけ・・・❤\nちょっとだけ歯型をつけて、それで許してあげる・・・❤\nがぶがぶ・・・♪,Don't worry; I won't bite too hard; just a little nibble...❤ Enough to leave a mark; and then I'll let you go...❤ Chomp chomp...♪
+あは、パンツに見惚れて、\nかみつかれるのと同時に射精しちゃった・・・❤,Haha; You were so entranced by my panties that you came right as I bit you...❤
+あなたの肩にわたしの歯型、\nくっきり残されちゃったね…❤\nほんのり痛くて、それで興奮するなんて、マゾなんだ。くすくす…❤,You left my teeth marks on your shoulder...❤Does it hurt? You seem to enjoy this ~❤ What a masochist. *Giggle*...❤
```
This feels a bit backwards to me, shouldn't it be "My teeth left an obvious mark on your shoulder"?
(the word 'obvious' is coming from くっきり "clear/distinct")

ArdynXT replied:
Not sure how I feel about that line. Personally I think we gotta rework it to something based on the infection possibly. I put your suggestion in for now.

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+だいじょうぶ、肉はかじりとらないで甘噛みするだけ・・・❤\nちょっとだけ歯型をつけて、それで許してあげる・・・❤\nがぶがぶ・・・♪,Don't worry; I won't bite too hard; just a little nibble...❤ Enough to leave a mark; and then I'll let you go...❤ Chomp chomp...♪
+あは、パンツに見惚れて、\nかみつかれるのと同時に射精しちゃった・・・❤,Haha; You were so entranced by my panties that you came right as I bit you...❤
+あなたの肩にわたしの歯型、\nくっきり残されちゃったね…❤\nほんのり痛くて、それで興奮するなんて、マゾなんだ。くすくす…❤,You left my teeth marks on your shoulder...❤Does it hurt? You seem to enjoy this ~❤ What a masochist. *Giggle*...❤
+ん、肩にがぶがぶされながら射精しちゃった・・・❤\n手でこすられて我慢できなかった・・・？\nそれとも噛まれたことに興奮しちゃった・・・？くすくす❤,You ejaculated while I bit down on your shoulder?❤ I couldn't resist rubbing it with my hand...❤ You taste so good...❤ *giggle*
```
Looks like the overall meaning got a bit mixed here. Suggesting changes since it flows into the next line

> You ejaculated while I bit down on your **shoulder...❤ You** couldn't resist **when my hand was stroking you...? Or perhaps you got excited from being bitten…?** *Giggle&ast;❤

These lines can be tough, since the JP doesn't actually specify whose hand is stroking who... it's all context based

ArdynXT replied:
I'll try and get the line to proc when I play and see the context but yea this is much better thank you!

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あは、パンツに見惚れて、\nかみつかれるのと同時に射精しちゃった・・・❤,Haha; You were so entranced by my panties that you came right as I bit you...❤
+あなたの肩にわたしの歯型、\nくっきり残されちゃったね…❤\nほんのり痛くて、それで興奮するなんて、マゾなんだ。くすくす…❤,You left my teeth marks on your shoulder...❤Does it hurt? You seem to enjoy this ~❤ What a masochist. *Giggle*...❤
+ん、肩にがぶがぶされながら射精しちゃった・・・❤\n手でこすられて我慢できなかった・・・？\nそれとも噛まれたことに興奮しちゃった・・・？くすくす❤,You ejaculated while I bit down on your shoulder?❤ I couldn't resist rubbing it with my hand...❤ You taste so good...❤ *giggle*
+その顔を見た感じ・・・、どっちもかな❤\nあなたの肩に、わたしの歯型、くっきり刻まれちゃったね・・・・❤,Quite the look on your face... I wonder if you enjoyed the bite more than I did; my teeth marks are now clearly etched on your shoulder. ・・・・❤
```
Suggesting change, since context is coming from previous line.

> **Judging by that look on your face... maybe it's both.❤** My teeth marks are now clearly etched on your shoulder...❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ん、肩にがぶがぶされながら射精しちゃった・・・❤\n手でこすられて我慢できなかった・・・？\nそれとも噛まれたことに興奮しちゃった・・・？くすくす❤,You ejaculated while I bit down on your shoulder?❤ I couldn't resist rubbing it with my hand...❤ You taste so good...❤ *giggle*
+その顔を見た感じ・・・、どっちもかな❤\nあなたの肩に、わたしの歯型、くっきり刻まれちゃったね・・・・❤,Quite the look on your face... I wonder if you enjoyed the bite more than I did; my teeth marks are now clearly etched on your shoulder. ・・・・❤
+ううっ・・・❤がまんできない・・・❤,Ugh...❤ I can't take it...❤
+ふふ・・・❤\nわたしでシコシコはじめちゃった・・・❤,I started masturbating.…
```
Dropped the laugh at the start, and this one needs to be flipped!

> Fufu…❤ **You've** started masturbating **because of me...❤**

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ううっ・・・❤がまんできない・・・❤,Ugh...❤ I can't take it...❤
+ふふ・・・❤\nわたしでシコシコはじめちゃった・・・❤,I started masturbating.…
+わたしの体、ゾンビになっているけれど\n女の子らしい部分がちゃんと残っているもんね・・・❤\n見てたらエッチな気分になっちゃうのも、くす、わかるよ・・・❤,I maybe a zombie; but my girlish parts are still here...❤ It's ok to feel naughty when you look at me...❤
+でも、戦闘中にシコシコしちゃうのは、\nダメだと思うな・・・❤\nほら隙だらけのあなたに近づいて・・・。,But I don't think it's good to stroke in the middle of a battle... ❤❤ I won't complain but; I am very hungry~❤
```
Second sentence needs a rework. If we translate it literally, it ends up being something like 
 "Look; I'm getting closer to you who's wide open"
(She's calling out why it is a bad idea to stroke in the middle of battle)

But we need to localize that somehow. To make sense with the first sentence, what about using:
"See how easily I can close in while you're vulnerable..."

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+でも、戦闘中にシコシコしちゃうのは、\nダメだと思うな・・・❤\nほら隙だらけのあなたに近づいて・・・。,But I don't think it's good to stroke in the middle of a battle... ❤❤ I won't complain but; I am very hungry~❤
+あなたの肩に、がぶり・・・❤,Lend me your shoulder darling ~❤
+はぐぅううっ・・・❤,Hagu ugh... ❤
+あは、噛み付かれて気持ちよくなっちゃってる・・・❤\nマゾなんだね・・・❤ふふ・・・❤,Haha; being bitten makes you feel so good... ❤ You are such a masochist... ❤Hmmmmm...❤
```
The "Hmmmmm...❤" at the end should be a "Fufu...❤"

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あなたの肩に、がぶり・・・❤,Lend me your shoulder darling ~❤
+はぐぅううっ・・・❤,Hagu ugh... ❤
+あは、噛み付かれて気持ちよくなっちゃってる・・・❤\nマゾなんだね・・・❤ふふ・・・❤,Haha; being bitten makes you feel so good... ❤ You are such a masochist... ❤Hmmmmm...❤
+ほら、噛まれたところから\nわたしのウイルスがどんどん入ってくる・・・❤\n元々感染してるのが、よけいにひどくなっちゃうね・・・❤,You see; my virus is infecting you...❤ The infection will become stronger and stronger…❤
```
Small missing context
 噛まれたところから = "from where I bit you"
 
 Could include with something like 
 "You see; **the** virus **from my bite** is infecting you...❤"
 
Or, if infection is the point here, we could go a bit more in depth with describing it
"Look; from where I bit you; my virus is steadily flowing into you…❤"
(normally I'd avoid using commas to break up a sentence like this, but the halting speech pattern is zombie-like... which is also why she uses so many ellipsis)

ArdynXT replied:
Alright I have changed it quite a bit do you think instead of "stronger and stronger" it should be "spreading throughout your body"?

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あは、噛み付かれて気持ちよくなっちゃってる・・・❤\nマゾなんだね・・・❤ふふ・・・❤,Haha; being bitten makes you feel so good... ❤ You are such a masochist... ❤Hmmmmm...❤
+ほら、噛まれたところから\nわたしのウイルスがどんどん入ってくる・・・❤\n元々感染してるのが、よけいにひどくなっちゃうね・・・❤,You see; my virus is infecting you...❤ The infection will become stronger and stronger…❤
+頭ぼーっとして、くらくらするでしょ・・・？\nでもシコシコする手、止められないんだ・・・❤\nわたしのことで頭いっぱいで、感染も気にならないのかな・・・❤,Are you feeling lightheaded and dizzy yet?❤ You must feel really good with me…❤  You don't even care about the infection...❤
+ほら、噛まれたところから\nわたしのウイルスがどんどん入ってくる・・・❤\n熱が広がって、体中が熱くなって、呼吸も荒くなっていくね・・・❤,You see; my virus is infecting you❤ You'll start to get feverish; and it'll become harder to breathe... ❤
```
I'd like to make a subtle change here from "it'll become harder to breathe" to "your breathing will get more ragged"

My reasoning is that the JP text seems to be ambiguously describing Zombie virus such that it could be interpreted as arousal, so I'd like to make the translation ambiguous if possible

ArdynXT replied:
That makes a lot of sense like an aphrodisiac of some kind

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ほら、噛まれたところから\nわたしのウイルスがどんどん入ってくる・・・❤\n元々感染してるのが、よけいにひどくなっちゃうね・・・❤,You see; my virus is infecting you...❤ The infection will become stronger and stronger…❤
+頭ぼーっとして、くらくらするでしょ・・・？\nでもシコシコする手、止められないんだ・・・❤\nわたしのことで頭いっぱいで、感染も気にならないのかな・・・❤,Are you feeling lightheaded and dizzy yet?❤ You must feel really good with me…❤  You don't even care about the infection...❤
+ほら、噛まれたところから\nわたしのウイルスがどんどん入ってくる・・・❤\n熱が広がって、体中が熱くなって、呼吸も荒くなっていくね・・・❤,You see; my virus is infecting you❤ You'll start to get feverish; and it'll become harder to breathe... ❤
+頭ぼーっとするでしょう？\nでもシコシコする手、止められないんだ・・・❤\nわたしのことで頭いっぱいで、感染も気にならないのかな・・・❤,It makes you light-headed; doesn't it? But you can't stop yourself… ❤I guess you are so preoccupied with me that you don't even care about the infection... ❤
```
But you can't stop yourself **from stroking...❤**

(If it reads weird, maybe replace "yourself" with "your hand"?
I don't know which sounds more natural... the JP uses "your hand" but "yourself" reads well to me.)

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ほら、噛まれたところから\nわたしのウイルスがどんどん入ってくる・・・❤\n熱が広がって、体中が熱くなって、呼吸も荒くなっていくね・・・❤,You see; my virus is infecting you❤ You'll start to get feverish; and it'll become harder to breathe... ❤
+頭ぼーっとするでしょう？\nでもシコシコする手、止められないんだ・・・❤\nわたしのことで頭いっぱいで、感染も気にならないのかな・・・❤,It makes you light-headed; doesn't it? But you can't stop yourself… ❤I guess you are so preoccupied with me that you don't even care about the infection... ❤
+あううっ・・・❤きもちいい・・・❤,Ohhhhhh...❤ So good...❤
+くすくす・・・❤\n朦朧としながらシコシコするの気持ちいい・・・？,*Giggle*... ❤ Does it feel good to get pleasured while your head is spinning?❤
```
get pleasured -> masturbate
 
 (This line and the ones around it occur when you select the 'masturbate' option against her)

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+頭ぼーっとするでしょう？\nでもシコシコする手、止められないんだ・・・❤\nわたしのことで頭いっぱいで、感染も気にならないのかな・・・❤,It makes you light-headed; doesn't it? But you can't stop yourself… ❤I guess you are so preoccupied with me that you don't even care about the infection... ❤
+あううっ・・・❤きもちいい・・・❤,Ohhhhhh...❤ So good...❤
+くすくす・・・❤\n朦朧としながらシコシコするの気持ちいい・・・？,*Giggle*... ❤ Does it feel good to get pleasured while your head is spinning?❤
+命が危なくなると、子孫を残すために、\nペニスが固くなるっていうけど、\nほんとみたい・・・❤,They say that when your life is in danger; your dick feels extra pleasure as a last resort to preserve your genes; Can you feel it?❤
```
Small change, but it changes the flow of the scene a lot:

> They say that when your life is in danger; your dick **gets hard** as a last resort to preserve your genes. **It seems like that's true...❤**

ArdynXT replied:
I like this a lot better!

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あううっ・・・❤きもちいい・・・❤,Ohhhhhh...❤ So good...❤
+くすくす・・・❤\n朦朧としながらシコシコするの気持ちいい・・・？,*Giggle*... ❤ Does it feel good to get pleasured while your head is spinning?❤
+命が危なくなると、子孫を残すために、\nペニスが固くなるっていうけど、\nほんとみたい・・・❤,They say that when your life is in danger; your dick feels extra pleasure as a last resort to preserve your genes; Can you feel it?❤
+噛まれる前よりガチガチに固くなって、\nシコシコするたびに顔がどんどん蕩けて、\nふふ・・・❤子孫を残すために、びゅってしたくてたまらないんだね❤,You are getting so hard;I think it’s even harder than before you were bitten; and every time I bite; That naughty expression on your face becomes even harder to hide. your cock is so eager to be sucked hoping to pass on it’s seed...❤❤
```
> It's become much harder than before you were bitten; and every time you stroke it; that naughty expression on your face becomes even harder to hide; fufu...❤ To ensure you leave offspring; you can't help wanting to cum; right?❤

Note: I'm taking some liberties in this suggestion, thoughts on using it?

ArdynXT replied:
No issues that is a much better interpretation and the flow is much better as well!

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+もちろんわたしにかけても子供なんてできないけど・・・❤\nでもそれでもいいよね・・・❤しこしこ気持ちいいもんね・・・❤\nくすくす❤,Of course it’s all for nothing; I can’t have kids...❤  *Giggle* ~❤ It feels so good to rub it in...❤
+あっ、あっ、ああっ・・・❤,Oh; oh; oh; oh...❤
+あひっ、あぐぅ、ああっ・・・❤❤,Ahh; ahhh; ahhh...❤❤❤
+ふふ・・・❤生命力をぜんぶ吐き出して、\n体が動かなくなっちゃった❤\nあなたの体、わたしの中に取り込んであげる・・・❤,I'm going to take your body into mine... ❤
```
Missing the first part!

> **Fufu...❤ You've expelled all your life force; and now your body can't move anymore~❤ Now** I'm going to take your body into mine...❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あひっ、あぐぅ、ああっ・・・❤❤,Ahh; ahhh; ahhh...❤❤❤
+ふふ・・・❤生命力をぜんぶ吐き出して、\n体が動かなくなっちゃった❤\nあなたの体、わたしの中に取り込んであげる・・・❤,I'm going to take your body into mine... ❤
+あは・・・❤\n強いんだね・・・❤,Haha... ❤ You are so lively… ❤
+別に一回ぐらいやられたって、\n何回でも蘇れるけど、ほら・・・。\n最後にちょっといいことしてあげる・・・❤,I can come back as many times as I want; It would be pointless to keep fighting me; especially when I can give you something nice…❤
```
I struggled a bit with this line, so my interpretation might be a bit off... but since she says this as her temptation if you drop her to one health, I think it is supposed to be:
> It's not a big deal if I get defeated once; I can be revived any number of times; but look... I'll do something nice for you before I go...❤

ArdynXT replied:
Yea this line felt very weird to me as well. I prefer the first half of mine but the second half of yours. I'll try and reword it let me know if this is any better!

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ふふ・・・❤生命力をぜんぶ吐き出して、\n体が動かなくなっちゃった❤\nあなたの体、わたしの中に取り込んであげる・・・❤,I'm going to take your body into mine... ❤
+あは・・・❤\n強いんだね・・・❤,Haha... ❤ You are so lively… ❤
+別に一回ぐらいやられたって、\n何回でも蘇れるけど、ほら・・・。\n最後にちょっといいことしてあげる・・・❤,I can come back as many times as I want; It would be pointless to keep fighting me; especially when I can give you something nice…❤
+わたしの胸、乳首が見えそうで、ずっと気になってたでしょ？\n見て・・・❤髪で隠れているのを横にどけて、\n冥途の土産にしっかり見せてあげる・・・❤,You’ve been trying to see my breasts and nipples this whole time right? Take a closer look...❤ my pink nipples hidden behind my hair; I can show you; consider it a parting gift till next time...❤
```
Thoughts on these changes for flow and clarity?

> You've been looking at my chest; wondering if you could catch a glimpse of my nipples; haven't you? Take a closer look...❤ I'll move aside my hair that's been hiding them; and show you properly as a parting gift till next time...❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あは・・・❤\n強いんだね・・・❤,Haha... ❤ You are so lively… ❤
+別に一回ぐらいやられたって、\n何回でも蘇れるけど、ほら・・・。\n最後にちょっといいことしてあげる・・・❤,I can come back as many times as I want; It would be pointless to keep fighting me; especially when I can give you something nice…❤
+わたしの胸、乳首が見えそうで、ずっと気になってたでしょ？\n見て・・・❤髪で隠れているのを横にどけて、\n冥途の土産にしっかり見せてあげる・・・❤,You’ve been trying to see my breasts and nipples this whole time right? Take a closer look...❤ my pink nipples hidden behind my hair; I can show you; consider it a parting gift till next time...❤
+見せるのは一瞬だけだから、\nじーって集中しないと見逃しちゃうかもね・・・❤\nくすくす・・・❤,It only takes a moment to show you; so you might miss it if you don't concentrate... ❤*Giggle*❤
```
It only takes a moment to show you -> I'll only show you for a moment

ArdynXT replied:
I was thinking "It'll only take a moment" as a way to try and convince the hero that it wouldn't be a big deal

ArdynXT replied:
I realized that in the context of the full sentence she is saying that if he blinks he'll miss it that definitely implies what you are saying I'll change it

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+誘惑を無視して攻撃した,Ignore the temptation and attack.
+胸に視線を吸い寄せられてしまった,My eyes were drawn to her breasts.
+きゃっ・・・。\nふふ、いいよ、また蘇ったら会おうね・・・❤,*Kya*… I'll see you when I revive again... ❤
+ふふ、釘付けになってる・・・❤\n早くとどめをさせばいいのに、間抜けな男・・・❤,Hmmm; Too bad looks like it’s over for you…❤ You should have finished me sooner dummy...❤
```
> **Fufu; you're completely transfixed...❤** You should have finished me sooner dummy...❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+胸に視線を吸い寄せられてしまった,My eyes were drawn to her breasts.
+きゃっ・・・。\nふふ、いいよ、また蘇ったら会おうね・・・❤,*Kya*… I'll see you when I revive again... ❤
+ふふ、釘付けになってる・・・❤\n早くとどめをさせばいいのに、間抜けな男・・・❤,Hmmm; Too bad looks like it’s over for you…❤ You should have finished me sooner dummy...❤
+隙だらけのあなたを、えい・・・。突き飛ばして・・・❤,You can't push me away… ❤
```
> Now that you're defenseless; ha...! I'll push you down...❤

(Tbh, I don't like using "ha...!" here. The original noise, "ei...!" has a cuter context. If you think of something else, feel free to add it instead.)

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ふふ、釘付けになってる・・・❤\n早くとどめをさせばいいのに、間抜けな男・・・❤,Hmmm; Too bad looks like it’s over for you…❤ You should have finished me sooner dummy...❤
+隙だらけのあなたを、えい・・・。突き飛ばして・・・❤,You can't push me away… ❤
+わたしの胸で挟んでイかせてあげる・・・❤,I'll sandwich you between my breasts and make you come... ❤
+うあっ・・・❤やめ、はなせっ・・・❤,Oh;❤ Stop; stop it...❤
```
I'm not the best at localizing these... thoughts on using the literal translation here?
> Uaa...❤ Sto- let go...❤

ArdynXT replied:
Yea that's much better!

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+命が危なくなると、子孫を残すために、\nペニスが固くなるっていうけど、\nほんとみたい・・・❤,They say that when your life is in danger; your dick feels extra pleasure as a last resort to preserve your genes; Can you feel it?❤
+噛まれる前よりガチガチに固くなって、\nシコシコするたびに顔がどんどん蕩けて、\nふふ・・・❤子孫を残すために、びゅってしたくてたまらないんだね❤,You are getting so hard;I think it’s even harder than before you were bitten; and every time I bite; That naughty expression on your face becomes even harder to hide. your cock is so eager to be sucked hoping to pass on it’s seed...❤❤
+ほら、あなたの子種汁、わたしのスカートにかけていいよ・・・❤\n女の子の「子供を作るところ」に、射精して、\nそれで自分の命を終わらせちゃおう・・・？,Here; you can pour your seed juice underneath my skirt...❤ Will you release on my "child making place" and end your life along with it...?
+もちろんわたしにかけても子供なんてできないけど・・・❤\nでもそれでもいいよね・・・❤しこしこ気持ちいいもんね・・・❤\nくすくす❤,Of course it’s all for nothing; I can’t have kids...❤  *Giggle* ~❤ It feels so good to rub it in...❤
```
Not sure if we should reword 41 and 42. It's a bit out of my wheelhouse, so I don't have any good suggestions.
Mentioning it for now, and will return if I come up with anything.

ArdynXT replied:
Agreed this one felt weird when translating it. I feel like with this fight in general these monsters have some undertones that are darker than other monsters/duelists.

Giratena replied:
I think the last sentence is the biggest offender here. The translation is wrong. She's saying the MC stroking feels good. Not her.

"But it feels so good to stroke your cock; doesn't it..."

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+隙だらけのあなたを、えい・・・。突き飛ばして・・・❤,You can't push me away… ❤
+わたしの胸で挟んでイかせてあげる・・・❤,I'll sandwich you between my breasts and make you come... ❤
+うあっ・・・❤やめ、はなせっ・・・❤,Oh;❤ Stop; stop it...❤
+くすくす・・・❤乳首見せてもらうだけのつもりだった？\nざんねんだけど、それだけじゃすまなかったね・・・❤,*Giggle*❤ And all you wanted was to see my nipples; I'm sorry; but I am not letting you go...❤
```
> *Giggle&ast;...❤ Did you think you'd get away with just seeing my nipples? Too bad; but things won't end with just that...❤

My suggestion might still be missing a bit of nuance here.

ArdynXT replied:
I took your suggestion maybe we can do "but that's not all" rather than "end with that"

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+わたしの胸で挟んでイかせてあげる・・・❤,I'll sandwich you between my breasts and make you come... ❤
+うあっ・・・❤やめ、はなせっ・・・❤,Oh;❤ Stop; stop it...❤
+くすくす・・・❤乳首見せてもらうだけのつもりだった？\nざんねんだけど、それだけじゃすまなかったね・・・❤,*Giggle*❤ And all you wanted was to see my nipples; I'm sorry; but I am not letting you go...❤
+あなたの弱点をわたしの胸で、\nすりすり、ずりずり・・・❤\n命のエナジーを吐き出させられちゃう・・・❤,I'm going to rub your weakest parts… ❤Don’t hold back I want to spill your seed… ❤
```
すりすり、ずりずり doesn't translate directly, so there's a bit of flexibility in how we do this line. 
How do you feel about:
> I'll rub and grind my breasts against your weak spot...❤ Making you spurt out your life energy...❤

Side Note on "Life Energy"
We've been translating エナジー to Mana, but now Zombie Girl is calling it 命のエナジー, and Life Mana doesn't sound quite right. I've been suggesting Life Energy so far, but we might need to replace it later, to keep consistent with Mana.

ArdynXT replied:
I believe all 3 of the monsters I've translated have had made mentions of life energy and mana. I defaulted to mana for the other 2 since I started to pick up more on the do's and don't's for translating. I'll leave it as life energy for now but worst comes to worst we can mass replace it with mana if needed.

HotHeadEmperor replied:
So I think 'life force' might fit best here. I know it's often like 'magical energy' or 'mana,' but i remember when I was doing nekomata, since she was a spirit, she used life force instead because-in lore-that's what they as creatures fed on. I think leaving it at for a zombie could fit too

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+くすくす・・・❤乳首見せてもらうだけのつもりだった？\nざんねんだけど、それだけじゃすまなかったね・・・❤,*Giggle*❤ And all you wanted was to see my nipples; I'm sorry; but I am not letting you go...❤
+あなたの弱点をわたしの胸で、\nすりすり、ずりずり・・・❤\n命のエナジーを吐き出させられちゃう・・・❤,I'm going to rub your weakest parts… ❤Don’t hold back I want to spill your seed… ❤
+でもほら、こうやって揺らしていると、\n乳首だって見えるでしょう？\nえ？見えない・・・？くすくす❤そっか、ざんねんだね・・・❤,You can’t see my nipples even when I'm rocking like this? *Giggle*❤ I see; that’s too bad...❤
+ううっ…あうっ❤おっぱい、きもちいいっ❤やわらかくてぇ・・・❤❤,Ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...
```
Looks like this line broke -  
> Ohhh… ahh.❤ Your boobs feel so good.❤ They're so soft...❤❤

ArdynXT replied:
Lmfao google translate XD

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+でもほら、こうやって揺らしていると、\n乳首だって見えるでしょう？\nえ？見えない・・・？くすくす❤そっか、ざんねんだね・・・❤,You can’t see my nipples even when I'm rocking like this? *Giggle*❤ I see; that’s too bad...❤
+ううっ…あうっ❤おっぱい、きもちいいっ❤やわらかくてぇ・・・❤❤,Ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...
+ふふ、髪の毛が乳首にぴったりくっついてるのは\n血でも乾いちゃったのかな・・・？\nでも、きもちいいなら、別にいいよね・・・❤,My hair is stuck to my chest due to the dried up blood... But you don’t care right? It still feels good...❤
+ゾンビになってもおっぱいはちゃんとやわらかいんだよ・・・❤\nひんやりはしてるかもしれないけど、\nそれがかえって気持ちいいかも・・・❤,My flesh might be cold but my boobs are still soft... ❤ Does the cool sensations feel good?❤
```
Context changed, she's implying that her paizuri might be better because she's cold.

> Even as a zombie; my boobs are still nice and soft...❤ They might be a bit cool to the touch; but that might actually make it feel better...❤

WisDZ reviewed (commented):
Review part 3!

Sorry it's taking me so long to review, I'll get this done eventually. (｡>﹏<)

I noticed that there's some spots where laughs are being translated to 'hmm' and a few places where meaning got changed or dropped. As general advice, I'd recommend trying to use multiple sources of translation where possible. Having multiple points of reference can help make sense of the lines.

On that note, if you have some spare time, could you pre-check the Fallen Angel and Grim Reaper cards against the translation bot on Gira's Discord? (No rush, though)

To use it, type "/translate" and select it.
<img width="464" height="224" alt="image" src="https://github.com/user-attachments/assets/74281572-9cd8-4be5-90b1-aa8667e4ce91" />

If you are shy about spamming the chat with translation requests, you can use the hidden:Yes option to have it reply to you privately.
<img width="540" height="64" alt="image" src="https://github.com/user-attachments/assets/99891089-4037-4a00-9bc2-7f0b9cdac389" />

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ううっ…あうっ❤おっぱい、きもちいいっ❤やわらかくてぇ・・・❤❤,Ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...ohhhh...
+ふふ、髪の毛が乳首にぴったりくっついてるのは\n血でも乾いちゃったのかな・・・？\nでも、きもちいいなら、別にいいよね・・・❤,My hair is stuck to my chest due to the dried up blood... But you don’t care right? It still feels good...❤
+ゾンビになってもおっぱいはちゃんとやわらかいんだよ・・・❤\nひんやりはしてるかもしれないけど、\nそれがかえって気持ちいいかも・・・❤,My flesh might be cold but my boobs are still soft... ❤ Does the cool sensations feel good?❤
+あなたの熱くなったペニスが、\nひんやりしたおっぱいに包まれて、\n激しく上下に、むにむに、ぐちゅぐちゅ・・・❤,Your hot flesh is wrapped in my cool tits and I’ll keep going up and down; up and down; up and down; up and down; mmmm; mmmm; mmmm...❤
```
I believe the translator broke on this one, 激しく上下に、むにむに、ぐちゅぐちゅ would have a literal translation close to "Violently up and down, squishy squishy, squelch squelch"
Since spoken sound effects can sound natural in JP but sound off in EN, we usually have to localize such lines. 

Something like this might work, though it's up to preference:
> Your hot penis is wrapped in my cool tits; while I move vigorously up and down; squeezing and sliding with wet; slippery pleasure...❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ふふ、髪の毛が乳首にぴったりくっついてるのは\n血でも乾いちゃったのかな・・・？\nでも、きもちいいなら、別にいいよね・・・❤,My hair is stuck to my chest due to the dried up blood... But you don’t care right? It still feels good...❤
+ゾンビになってもおっぱいはちゃんとやわらかいんだよ・・・❤\nひんやりはしてるかもしれないけど、\nそれがかえって気持ちいいかも・・・❤,My flesh might be cold but my boobs are still soft... ❤ Does the cool sensations feel good?❤
+あなたの熱くなったペニスが、\nひんやりしたおっぱいに包まれて、\n激しく上下に、むにむに、ぐちゅぐちゅ・・・❤,Your hot flesh is wrapped in my cool tits and I’ll keep going up and down; up and down; up and down; up and down; mmmm; mmmm; mmmm...❤
+ふふふ、気持ちよすぎて、腰が震えるの、抑えられないね・・・❤\nあとちょっとで倒せそうだったのに、\nおっぱいに負けそうになってる・・・❤,Your hips won’t stop shaking...❤ you had me at your mercy; but now you are succumbing to my tits… ❤
```
Missing a few bits of the original line:
> **Fufufu; it feels so good that** your hips are trembling. **You can't hold back; can you...❤** You had me at your mercy; but now you are succumbing to my tits...❤"

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あは、その目・・・❤まだ乳首が見たいんだ・・・❤\nおっぱいが揺れるの必死になって見て・・・、\n男ってわかりやすい・・・❤,Haha; you look frustrated... ❤ Still trying to see my nipples?❤ The desperation as you watch my tits bounce;❤ men are such simple creatures…❤
+がんばって射精をこらえれば、乳首見れるかもよ・・・？\n激しくこすられて、我慢できれば、だけど・・・❤,Maybe if you can last a little longer; you could get a glimpse...❤ Do you think you have a chance when I am bouncing like this?❤
+あっ、あっ、あああっ❤❤❤,Oh; oh; oh ❤❤❤❤;.
+くす・・・❤やっぱり耐えるのは無理みたい・・・❤\nさきっぽをこすったとき、\n特に腰が動いてるから、ここが一番いいのかな・・・？,I guess you won’t last after all...❤
```
> **&ast;Giggle&ast;…❤** I guess you won't last after all…❤ **When I rub the tip; your hips move so much... I wonder if this spot feels the best…?**

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あっ、あっ、あああっ❤❤❤,Oh; oh; oh ❤❤❤❤;.
+くす・・・❤やっぱり耐えるのは無理みたい・・・❤\nさきっぽをこすったとき、\n特に腰が動いてるから、ここが一番いいのかな・・・？,I guess you won’t last after all...❤
+先端がおっぱいにむにゅうって挟みこまれて、\nおちんちん溶けちゃいそう？\nふふ、どろどろに命を溶かして、ここから出しちゃっていいんだよ・・・❤,I’m going to start squeezing the tip nice and tight; Do you think you can handle it? Melt into mush and let it out here...❤
+わたしに、命のエナジーをちょうだい・・・？\nほら、ほら、ほら・・・❤,Bathe my chest in your seed...❤ Here; here; here; here... ❤
```
We might want to localize "Hora; hora; hora...❤" differently

Instead of "here; here; here..." I think we can use something that describes the scene
(I'm using additional ellipsis in the examples to match halting Zombie-speak, but you can edit these or make your own replacement)

> Feel how I'm coaxing it out... with every movement...❤

> As I squeeze and stroke with my breasts... again and again...❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+先端がおっぱいにむにゅうって挟みこまれて、\nおちんちん溶けちゃいそう？\nふふ、どろどろに命を溶かして、ここから出しちゃっていいんだよ・・・❤,I’m going to start squeezing the tip nice and tight; Do you think you can handle it? Melt into mush and let it out here...❤
+わたしに、命のエナジーをちょうだい・・・？\nほら、ほら、ほら・・・❤,Bathe my chest in your seed...❤ Here; here; here; here... ❤
+あぐっ、はひぃ・・・・❤❤,Agghh; hahi・・・・❤❤❤;
+ふふ、おっぱいがあったかい・・・❤\nあなたの出したエネルギーで、\nひんやりしてた胸があったまっちゃいそう・・・❤,My breasts are so warm...❤I feel your energy seeping into my un-dead bossom...❤
```
Small edit for clarity:

> **Fufu;** my breasts are so warm **now...❤** I feel your energy seeping into my un-dead bosom...❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+このエネルギーを吸収して、体力も回復・・・❤\n一回分、復活する手間がはぶけちゃった。\n命をありがとう、おいしかったよ・・・❤くすくす・・・❤,So much Magical energy; I feel my strength restored...❤ Looks like I can save my master the trouble of bringing me back...Thank you for the meal; it was delicious...❤ *giggle*❤
+くすくす…❤わたしのここに精液かけられて、満足した・・・？\nオナニーで負けちゃうなんて、へんたいだね・・・❤,*Giggle*...❤ Are you satisfied with your seed being spilled here? I didn’t even have to touch you...❤
+乳首が見れなくて残念だったね…❤\nでも、わたしのおっぱい気持ちよかったなら、\nそれで十分でしょ。くすくす…❤,Too bad you couldn't see my nipples...❤ But if my tits felt that good; then I’m sure you can’t complain *Giggle*❤
+んっ・・・❤見て、こんなにたくさん出しちゃったね・・・❤\n生気にあふれてて、おいしい・・・❤ごくん・・・❤,Look how much you’ve cum...❤ Lush and delicious...❤ Gasp...❤
```
Recommending some changes:

> **Mmm…❤** Look how much you’ve cum...❤ So delicious **and full of life...❤ I'll savor every drop...❤**

Note: "I'll savor every drop" is my attempt at localizing "*Gulp&ast;...❤"
can localize differently if you like

ArdynXT replied:
I like this much better! I will implement it now

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あうう・・・。,Ahhhh!
+もう逃げられないね・・・❤\nあなたの負けだよ、正義のカードマスターくん・・・❤,You can't get away now... ❤ You lose; Mr. Righteous Cardmaster... ❤
+ほら、カラカラになるまで搾り取ってあげる・・・❤\nホントはお肉を食べてしまいたいけど・・・。,Here; I'll squeeze you until you're dry… ❤ I really want to take a bite; but.... 
+ダメよ。わたし殺しはやらない主義なの。分かってるでしょ。,No; you can't eat him! I'm a not a killer! But you may pleasure him to restore my mana.
```
This is Necro speaking, I _think_ it's supposed to a firm tone, rather than a panicked one

> No; you can't eat him. **I have a policy against killing; you know that already.**

Alternative options, mix and match if you want:
"Don't even think about it. I have a strict no-killing policy; you should know that by now."
"Absolutely not. My rule is no killing; you're well aware of that."

ArdynXT replied:
I took some liberties here and did a mishmash of a couple I think it should sound urgent but firm. Let me know how this looks

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+もう逃げられないね・・・❤\nあなたの負けだよ、正義のカードマスターくん・・・❤,You can't get away now... ❤ You lose; Mr. Righteous Cardmaster... ❤
+ほら、カラカラになるまで搾り取ってあげる・・・❤\nホントはお肉を食べてしまいたいけど・・・。,Here; I'll squeeze you until you're dry… ❤ I really want to take a bite; but.... 
+ダメよ。わたし殺しはやらない主義なの。分かってるでしょ。,No; you can't eat him! I'm a not a killer! But you may pleasure him to restore my mana.
+ふふ、マスターのやさしさに感謝してね・・・。\nでも死ぬ少し手前まで、たっぷり搾り取ってあげる・・・❤,Thank you for your kindness Master.... I'll keep squeezing you for your precious mana...❤
```
& then Zombie Girl is the speaker again.
Recommending changes to flow with the previous line

> Fufu; be grateful for Master's kindness… But I'll still thoroughly drain you until you're just short of dying…❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ふふ、マスターのやさしさに感謝してね・・・。\nでも死ぬ少し手前まで、たっぷり搾り取ってあげる・・・❤,Thank you for your kindness Master.... I'll keep squeezing you for your precious mana...❤
+うう、やめろ・・・。盗賊になんて負けない・・。,Ugh; stop... I will not be defeated by a bandit…
+悪いことはやめさせる・・・。僕はBランクだから・・。,I won’t let you have your way… I'm a B rank...
+ふふふ・・・❤,Hmmm... ❤
```
This is another laugh instead of a hmm.
"Fufufu...❤"

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+悪いことはやめさせる・・・。僕はBランクだから・・。,I won’t let you have your way… I'm a B rank...
+ふふふ・・・❤,Hmmm... ❤
+あっ、あっ、ああっ❤❤,Oh; oh; oh...❤❤
+止めたければ、わたしみたいな下級モンスターぐらいには勝たないと・・・❤\n何度も蘇ってきて、何度も誘惑されても、\n我慢しないとダメ・・・❤,If you want to stop us; you just have to beat a low level monster like me...❤ Just resist my temptation as I tease you over and over again...❤
```
I'd like to match the repeated "何度も" being used, how does this sound?

> If you want to stop us; you'll need to at least be able to defeat a low-level monster like me…❤ Even if I keep coming back to life again and again... Even if you're tempted over and over... You have to resist…❤

ArdynXT replied:
This is much better!

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ふふふ・・・❤,Hmmm... ❤
+あっ、あっ、ああっ❤❤,Oh; oh; oh...❤❤
+止めたければ、わたしみたいな下級モンスターぐらいには勝たないと・・・❤\n何度も蘇ってきて、何度も誘惑されても、\n我慢しないとダメ・・・❤,If you want to stop us; you just have to beat a low level monster like me...❤ Just resist my temptation as I tease you over and over again...❤
+責められておちんちんに段々ムラムラが貯まっていっても・・・、\n負けないで我慢して、びゅるるぅってせずに、戦うの・・・。,Even if the shame only increases your arousal...don't give up you just have to fight off these naughty thoughts.❤
```
Minor edit for flow:
> Even if the shame only increases your arousal; **don't give up...** You just have to fight off these naughty thoughts....

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あっ、あっ、ああっ❤❤,Oh; oh; oh...❤❤
+止めたければ、わたしみたいな下級モンスターぐらいには勝たないと・・・❤\n何度も蘇ってきて、何度も誘惑されても、\n我慢しないとダメ・・・❤,If you want to stop us; you just have to beat a low level monster like me...❤ Just resist my temptation as I tease you over and over again...❤
+責められておちんちんに段々ムラムラが貯まっていっても・・・、\n負けないで我慢して、びゅるるぅってせずに、戦うの・・・。,Even if the shame only increases your arousal...don't give up you just have to fight off these naughty thoughts.❤
+あなたみたいに、おっぱいで挟まれて、\nあんあん喘いでいるだけじゃ、ダメ・・・❤\nそんな強さじゃ、なにも止められないよ・・・❤,If you weren’t trapped between my tits you could easily overpower me...❤ You just have to resist these urges...❤
```
The previous lines were "supportive", but this one seems to be more mocking. Does this work as a localization?

> But just look at you; trapped between my breasts; moaning and gasping helplessly...❤ With such pathetic resistance; you can't stop anything...❤

ArdynXT replied:
I like your changes better. I would argue for my translation that she is a very weak monster and the only reason the player is in this position is because he wants to be but, again I like the flow of yours much better

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+止めたければ、わたしみたいな下級モンスターぐらいには勝たないと・・・❤\n何度も蘇ってきて、何度も誘惑されても、\n我慢しないとダメ・・・❤,If you want to stop us; you just have to beat a low level monster like me...❤ Just resist my temptation as I tease you over and over again...❤
+責められておちんちんに段々ムラムラが貯まっていっても・・・、\n負けないで我慢して、びゅるるぅってせずに、戦うの・・・。,Even if the shame only increases your arousal...don't give up you just have to fight off these naughty thoughts.❤
+あなたみたいに、おっぱいで挟まれて、\nあんあん喘いでいるだけじゃ、ダメ・・・❤\nそんな強さじゃ、なにも止められないよ・・・❤,If you weren’t trapped between my tits you could easily overpower me...❤ You just have to resist these urges...❤
+さっき射精したのに、もうこんなに固い・・・❤\n敵の魔物のおっぱいで、すりすりされただけで、\nすぐ気持ちよくて動けなくなる・・・❤,You just came a few minutes ago and yet; it’s already so hard…❤ Just being rubbed by the enemies tits is enough to paralyze you…❤
```
small change rec
minutes -> moments

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+責められておちんちんに段々ムラムラが貯まっていっても・・・、\n負けないで我慢して、びゅるるぅってせずに、戦うの・・・。,Even if the shame only increases your arousal...don't give up you just have to fight off these naughty thoughts.❤
+あなたみたいに、おっぱいで挟まれて、\nあんあん喘いでいるだけじゃ、ダメ・・・❤\nそんな強さじゃ、なにも止められないよ・・・❤,If you weren’t trapped between my tits you could easily overpower me...❤ You just have to resist these urges...❤
+さっき射精したのに、もうこんなに固い・・・❤\n敵の魔物のおっぱいで、すりすりされただけで、\nすぐ気持ちよくて動けなくなる・・・❤,You just came a few minutes ago and yet; it’s already so hard…❤ Just being rubbed by the enemies tits is enough to paralyze you…❤
+ふふふ、かわいいじゃない❤\n男の子なんてそれぐらいでいいのよ❤,Hmmm; you're so cute ❤; isn’t this all a boy needs?❤
```
This is Necro chiming in, I'm not clear on if she's meant to be addressing the player or Zombie Girl

assuming she's talking to the player:
> Fufufu; aren't you just adorable~❤ This is exactly what boys should be like.❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+さっき射精したのに、もうこんなに固い・・・❤\n敵の魔物のおっぱいで、すりすりされただけで、\nすぐ気持ちよくて動けなくなる・・・❤,You just came a few minutes ago and yet; it’s already so hard…❤ Just being rubbed by the enemies tits is enough to paralyze you…❤
+ふふふ、かわいいじゃない❤\n男の子なんてそれぐらいでいいのよ❤,Hmmm; you're so cute ❤; isn’t this all a boy needs?❤
+ほら、わたしも{自分}のお手伝い。\n顔におっぱいを乗せてあげる❤,Here; I'll help out {自分}; by putting my boobs on your face❤
+盗賊の用心棒のおっぱいなんかに負けちゃダメよ。\nコノハちゃんの弟子くん❤\nえい❤,Look at you losing yourself in the embrace of a bandits chest.
```
Still Necro speaking. She's saying "thief's bodyguard's breasts" here, but it's a bit wordy...
 
 How does this look?
> Don't lose to the breasts of a mere bodyguard; Konoha's little apprentice~❤ Take this~❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ほら、わたしも{自分}のお手伝い。\n顔におっぱいを乗せてあげる❤,Here; I'll help out {自分}; by putting my boobs on your face❤
+盗賊の用心棒のおっぱいなんかに負けちゃダメよ。\nコノハちゃんの弟子くん❤\nえい❤,Look at you losing yourself in the embrace of a bandits chest.
+あう、あうううっ・・・❤❤❤,Oh; ahhhhh...❤❤❤❤
+ふふ❤わたしのおっぱいやわらかいでしょう？\n{自分}のと比べるとあったかいから、\n余計にふわふわに感じちゃうかも❤,My boobs are soft; aren't they? They are much warmer than my {自分}; so they might feel even fluffier.
```
Add heart to the end and 'Fufu~❤" at the start

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+盗賊の用心棒のおっぱいなんかに負けちゃダメよ。\nコノハちゃんの弟子くん❤\nえい❤,Look at you losing yourself in the embrace of a bandits chest.
+あう、あうううっ・・・❤❤❤,Oh; ahhhhh...❤❤❤❤
+ふふ❤わたしのおっぱいやわらかいでしょう？\n{自分}のと比べるとあったかいから、\n余計にふわふわに感じちゃうかも❤,My boobs are soft; aren't they? They are much warmer than my {自分}; so they might feel even fluffier.
+おっぱいに埋もれて表情は見えないけど・・・❤\n腕も脚も完全に力が抜けてる・・・❤\nふふ、きっと幸せそうな顔になっているんでしょうね❤,I can't see his expression because he’s buried in your boobs... ❤❤ his arms and legs are completely relaxed...❤ Hmmm; he must be grinning ear to ear.❤
```
Another Hmm -> Fufu

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あう、あうううっ・・・❤❤❤,Oh; ahhhhh...❤❤❤❤
+ふふ❤わたしのおっぱいやわらかいでしょう？\n{自分}のと比べるとあったかいから、\n余計にふわふわに感じちゃうかも❤,My boobs are soft; aren't they? They are much warmer than my {自分}; so they might feel even fluffier.
+おっぱいに埋もれて表情は見えないけど・・・❤\n腕も脚も完全に力が抜けてる・・・❤\nふふ、きっと幸せそうな顔になっているんでしょうね❤,I can't see his expression because he’s buried in your boobs... ❤❤ his arms and legs are completely relaxed...❤ Hmmm; he must be grinning ear to ear.❤
+おちんちんも目で見て分かるぐらい固くなってるわ❤\n{自分}の胸の中からこぼれそうなぐらい、びくびく跳ねてる❤,Your cock is so hard~❤; and it's bouncing around so much...❤ It's about to spill right on my chest.
```
Still Necro talking here, swapping to second person and adding a bit of detail that got dropped
> Your penis has gotten so hard **I can see it clearly.❤** It's twitching and jumping so much; **it looks like it might spill out from {自分}'s cleavage.❤**

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ふふ❤わたしのおっぱいやわらかいでしょう？\n{自分}のと比べるとあったかいから、\n余計にふわふわに感じちゃうかも❤,My boobs are soft; aren't they? They are much warmer than my {自分}; so they might feel even fluffier.
+おっぱいに埋もれて表情は見えないけど・・・❤\n腕も脚も完全に力が抜けてる・・・❤\nふふ、きっと幸せそうな顔になっているんでしょうね❤,I can't see his expression because he’s buried in your boobs... ❤❤ his arms and legs are completely relaxed...❤ Hmmm; he must be grinning ear to ear.❤
+おちんちんも目で見て分かるぐらい固くなってるわ❤\n{自分}の胸の中からこぼれそうなぐらい、びくびく跳ねてる❤,Your cock is so hard~❤; and it's bouncing around so much...❤ It's about to spill right on my chest.
+くす・・・❤捕まえた獲物は逃がさない・・・❤\nそんなにおちんちん暴れさせても、\nこのおっぱいからは逃げられないよ・・・❤,You can't escape from my tits no matter how much you struggle.…
```
Finally a Zombie Girl line, I think that some detail was lost here:
"Hehe…❤ I never let captured prey escape…❤ Even if you make your penis thrash about so much; you can't escape from these breasts…❤"

(I didn't put this into a code break line, since I think my suggestion might need some localization still - feel free to edit any of my suggestions to sound more natural, but especially this one 😅)

ArdynXT replied:
Made a hybrid translation feel free to take a look I think it is good!

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+おっぱいに埋もれて表情は見えないけど・・・❤\n腕も脚も完全に力が抜けてる・・・❤\nふふ、きっと幸せそうな顔になっているんでしょうね❤,I can't see his expression because he’s buried in your boobs... ❤❤ his arms and legs are completely relaxed...❤ Hmmm; he must be grinning ear to ear.❤
+おちんちんも目で見て分かるぐらい固くなってるわ❤\n{自分}の胸の中からこぼれそうなぐらい、びくびく跳ねてる❤,Your cock is so hard~❤; and it's bouncing around so much...❤ It's about to spill right on my chest.
+くす・・・❤捕まえた獲物は逃がさない・・・❤\nそんなにおちんちん暴れさせても、\nこのおっぱいからは逃げられないよ・・・❤,You can't escape from my tits no matter how much you struggle.…
+ほらほらほら・・・・❤\nおっぱいでもぐもぐされて、おちんちんの中身全部食べられちゃえ・・・❤,Here; here; here・・・・❤❤❤; give into the pleasure of our tits and feed us your mana... ❤
```
In this context, ほらほらほら (hora hora hora) is probably localized best as an encouraging phrase.
Also changed the second phrase to capture some context (the もぐもぐ bit, specifically) 

Thoughts on using:
> Come on; come on; that's it...❤ Get devoured by my breasts; and let them drain everything out of your cock...❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+おちんちんも目で見て分かるぐらい固くなってるわ❤\n{自分}の胸の中からこぼれそうなぐらい、びくびく跳ねてる❤,Your cock is so hard~❤; and it's bouncing around so much...❤ It's about to spill right on my chest.
+くす・・・❤捕まえた獲物は逃がさない・・・❤\nそんなにおちんちん暴れさせても、\nこのおっぱいからは逃げられないよ・・・❤,You can't escape from my tits no matter how much you struggle.…
+ほらほらほら・・・・❤\nおっぱいでもぐもぐされて、おちんちんの中身全部食べられちゃえ・・・❤,Here; here; here・・・・❤❤❤; give into the pleasure of our tits and feed us your mana... ❤
+まだまだ出るでしょ？ふふ・・・❤\n生命エネルギー、どんどんちょうだい・・・❤,You're still coming; aren't you? Give me more of your life energy... ❤
```
Small edits:
> You can still cum more; can't you? Fufu…❤ Give me more **and more** of your life energy…❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ほらほらほら・・・・❤\nおっぱいでもぐもぐされて、おちんちんの中身全部食べられちゃえ・・・❤,Here; here; here・・・・❤❤❤; give into the pleasure of our tits and feed us your mana... ❤
+まだまだ出るでしょ？ふふ・・・❤\n生命エネルギー、どんどんちょうだい・・・❤,You're still coming; aren't you? Give me more of your life energy... ❤
+おっぱいがどろどろ・・・❤\nわたしのひんやりした体には、\nこの精液、とても温かく感じる・・・❤,My tits are covered...❤The warmth of your cum feels so good against my cold chest...❤
+ねえ、いまおっぱいから解放されて、自由だよ・・・❤\nまだ干からびるほどは搾ってないから、一応動けると思うけど・・・。,What’s wrong B rank master; I’ve let you go but you are still right where I left you...❤ 
```
> Hey; I'm releasing you from my breasts now; you're free...❤ I haven't drained you to the point of shriveling up yet; so I think you can still move somewhat...

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+まだまだ出るでしょ？ふふ・・・❤\n生命エネルギー、どんどんちょうだい・・・❤,You're still coming; aren't you? Give me more of your life energy... ❤
+おっぱいがどろどろ・・・❤\nわたしのひんやりした体には、\nこの精液、とても温かく感じる・・・❤,My tits are covered...❤The warmth of your cum feels so good against my cold chest...❤
+ねえ、いまおっぱいから解放されて、自由だよ・・・❤\nまだ干からびるほどは搾ってないから、一応動けると思うけど・・・。,What’s wrong B rank master; I’ve let you go but you are still right where I left you...❤ 
+戦ったり逃げたりしないの・・・？\n悪いことはやめさせるんじゃなかった・・・？\nBランクマスターくん・・・❤,Have you lost your resolve? I thought you were going to stop me from doing bad things...? Are you a Boob lover B rank master? ❤
```
> Aren't you going to fight or run away...? I thought you were going to stop me from doing bad things...? Little B-Rank Master~❤

ArdynXT replied:
I personally like lost your resolve over fight or run. and boob lover b rank felt more appropriate for the mocking rather just Little.... Thoughts?

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ねえ、いまおっぱいから解放されて、自由だよ・・・❤\nまだ干からびるほどは搾ってないから、一応動けると思うけど・・・。,What’s wrong B rank master; I’ve let you go but you are still right where I left you...❤ 
+戦ったり逃げたりしないの・・・？\n悪いことはやめさせるんじゃなかった・・・？\nBランクマスターくん・・・❤,Have you lost your resolve? I thought you were going to stop me from doing bad things...? Are you a Boob lover B rank master? ❤
+あうう・・・・❤❤きもちいい・・・❤\nおっぱい・・・❤❤おっぱいぃ・・・❤❤,Ahhhhhh・・・・❤❤❤ You feel so good...❤❤❤ boobs...❤❤ boobs...❤❤❤
+ふふ、パイズリは終わってるのに、\nうわごとを言ってる・・・。\nおちんちんにまだパイズリされた感触が残っているのかな・・・❤,I wonder if he still remembers the pleasure of being beaten by my breasts... ❤
```
Dropped the first sentence here:

> **Fufu; even though the titjob is over; you're still mumbling deliriously...** I wonder if **your penis** still remembers the pleasure of being beaten by my breasts...❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ウイルスをうつしてあげるまでもなく、\n簡単に脳みそ壊れちゃったね・・・❤\nくすくす❤,I don't even need to give you the virus; your brain is already ruined from the pleasure... ❤*Giggle*❤
+その方が楽でいいわ。ゾンビにしてしまうと、\n普通の人間に戻すのが大変だから・・・。,It's easier this way. If you turn them into zombies; it's hard to turn them back into humans.... 
+それにしてもBランクマスターか。\nその若さですごいのね。急に連れ去ったりしたら騒ぎになるかも？,But still; a B-rank master? That's amazing at such a young age. If he disappeared it might cause a commotion.
+このあとどうしようかしら。\n牢屋に入れておくか、それともいったん解放してしまうか・・・。\nうーん・・・。,What should we do after this? Should I keep him prisoner or let him go?
```
The うーん・・・。 at the end should probably be a "Hmm..." in the EN

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+それにしてもBランクマスターか。\nその若さですごいのね。急に連れ去ったりしたら騒ぎになるかも？,But still; a B-rank master? That's amazing at such a young age. If he disappeared it might cause a commotion.
+このあとどうしようかしら。\n牢屋に入れておくか、それともいったん解放してしまうか・・・。\nうーん・・・。,What should we do after this? Should I keep him prisoner or let him go?
+その後、僕は盗賊の根城から解放され、出口に放り出された。,I was then freed from the bandit's stronghold and thrown out the exit.
+もはやカードを使う魔力も残っておらず、僕はふらふらとした足取りで村に戻っていくほかない。侵入するとき僕が気絶させていた盗賊たちは目を覚まし、立場が逆転した僕の様子を見ながら、にやにやと根城から出ていくのを見送ったのだった。,No longer having any magic power left to use my cards; I had no choice but to wobble back to the village. The bandits I had stunned during the break-in woke up and watched me with a grin as I walked out of the castle.
```
Given that the bandits are 'waking up' here, maybe we should use "knocked out" instead of "stunned"
Thoughts?

ArdynXT replied:
Much better I missed  this completely!

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+それから数日後。,Then a few days later.
+魔力を回復させ、僕は再度盗賊討伐に挑戦した。坑道の入口まで行くと、今度は盗賊たちは一切抵抗せず、僕を奥の部屋へと案内した。,After recovering my magical power; I tried going back to defeat the bandits. But when I reached the entrance of the tunnel; this time the bandits did not put up any resistance and led me to a back room.
+その部屋には盗賊たちの用心棒のネクロがいて、そしてまたバトルになり・・・。,In that room was Necro; another battle ensued.
+くす・・・❤わたしを１回も倒せないで負けちゃうね・・・❤\n何回も復活するから強いのに、\n今日はただ普通におっぱいで挟まれただけで負けちゃう・・・・❤,*Kusu*...❤ You couldn’t even put up a fight...❤ Last time I had to keep coming back to tear at your resolve; but this time you got practically threw yourself into my tits and lost・・・・❤
```
*Kusu*  ->  *Giggle*

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+魔力を回復させ、僕は再度盗賊討伐に挑戦した。坑道の入口まで行くと、今度は盗賊たちは一切抵抗せず、僕を奥の部屋へと案内した。,After recovering my magical power; I tried going back to defeat the bandits. But when I reached the entrance of the tunnel; this time the bandits did not put up any resistance and led me to a back room.
+その部屋には盗賊たちの用心棒のネクロがいて、そしてまたバトルになり・・・。,In that room was Necro; another battle ensued.
+くす・・・❤わたしを１回も倒せないで負けちゃうね・・・❤\n何回も復活するから強いのに、\n今日はただ普通におっぱいで挟まれただけで負けちゃう・・・・❤,*Kusu*...❤ You couldn’t even put up a fight...❤ Last time I had to keep coming back to tear at your resolve; but this time you got practically threw yourself into my tits and lost・・・・❤
+ふふ、王都に戻って応援でも呼んで来ればいいのに。\nすぐここに来たのは、あなたのプライドのせい？\nそれとも・・・・。,Hmmm; why didn't you just go back to the guild and call for backup? Did you come here right away to protect the last bits of your pride? Or is it ・・・・
```
Hmmm -> Fufu

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+その部屋には盗賊たちの用心棒のネクロがいて、そしてまたバトルになり・・・。,In that room was Necro; another battle ensued.
+くす・・・❤わたしを１回も倒せないで負けちゃうね・・・❤\n何回も復活するから強いのに、\n今日はただ普通におっぱいで挟まれただけで負けちゃう・・・・❤,*Kusu*...❤ You couldn’t even put up a fight...❤ Last time I had to keep coming back to tear at your resolve; but this time you got practically threw yourself into my tits and lost・・・・❤
+ふふ、王都に戻って応援でも呼んで来ればいいのに。\nすぐここに来たのは、あなたのプライドのせい？\nそれとも・・・・。,Hmmm; why didn't you just go back to the guild and call for backup? Did you come here right away to protect the last bits of your pride? Or is it ・・・・
+わたしたちのおっぱいを忘れられなかった？\nまあ戻ってくると思ったから解放したんだけど。\nくすくす❤,You couldn't forget our tits? I knew you’d come back; so I let you go. *Giggle*❤
```
Change middle sentence for clarity? "Well; I released you because I thought you would come back." is roughly the sentence, but I think it can be localized like:
 
> You couldn't forget our tits? **Well;** I knew you’d come back; **that's why I let you go in the first place.** *Giggle&ast;❤

("Well" came from まあ, but if it doesn't flow well, feel free to drop it)

ArdynXT replied:
I see this looks a lot better and makes sense for necro

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+くす・・・❤わたしを１回も倒せないで負けちゃうね・・・❤\n何回も復活するから強いのに、\n今日はただ普通におっぱいで挟まれただけで負けちゃう・・・・❤,*Kusu*...❤ You couldn’t even put up a fight...❤ Last time I had to keep coming back to tear at your resolve; but this time you got practically threw yourself into my tits and lost・・・・❤
+ふふ、王都に戻って応援でも呼んで来ればいいのに。\nすぐここに来たのは、あなたのプライドのせい？\nそれとも・・・・。,Hmmm; why didn't you just go back to the guild and call for backup? Did you come here right away to protect the last bits of your pride? Or is it ・・・・
+わたしたちのおっぱいを忘れられなかった？\nまあ戻ってくると思ったから解放したんだけど。\nくすくす❤,You couldn't forget our tits? I knew you’d come back; so I let you go. *Giggle*❤
+ねえ、コノハちゃんの弟子くん？\nわたしからあなたにおねがいがあるの。,Hey; Konoha's disciple, I have a favor to ask you.
```
I like maintaining the question mark here:
> Hey; Konoha's disciple? I have a favor to ask you.

Actually, コノハちゃんの弟子くん (Konoha-chan's disciple-kun) is tough... maybe we could use something like 
"Little Apprentice" or some other nickname.

ArdynXT replied:
Added the question mark and changed it to little. At this point it would make sense for her to feel even more superior to him given the context

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ふふ、王都に戻って応援でも呼んで来ればいいのに。\nすぐここに来たのは、あなたのプライドのせい？\nそれとも・・・・。,Hmmm; why didn't you just go back to the guild and call for backup? Did you come here right away to protect the last bits of your pride? Or is it ・・・・
+わたしたちのおっぱいを忘れられなかった？\nまあ戻ってくると思ったから解放したんだけど。\nくすくす❤,You couldn't forget our tits? I knew you’d come back; so I let you go. *Giggle*❤
+ねえ、コノハちゃんの弟子くん？\nわたしからあなたにおねがいがあるの。,Hey; Konoha's disciple, I have a favor to ask you.
+あなたが満足するまでわたしたちに挑戦したら、\nあなたはギルドに「盗賊はもう退治しました」って報告してほしいの❤\nこんなに気持ちよくしてあげてるだもん。それぐらいいいでしょ？,Keep challenging us until you are satisfied; In exchange I want you to report to the guild that you have killed the bandits. That's good enough for you; isn't it?
```
killed should probably be defeated here, since earlier text made a point about stunning or knocking out the bandits.
Last sentence is like "I'm making you feel this good; so that much should be fine; right?"

> Keep challenging us until you are satisfied; In exchange I want you to report to the guild that you have **defeated** the bandits. **Since I'm making you feel so good; this little favor isn't too much to ask; is it?**

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+わたしたちのおっぱいを忘れられなかった？\nまあ戻ってくると思ったから解放したんだけど。\nくすくす❤,You couldn't forget our tits? I knew you’d come back; so I let you go. *Giggle*❤
+ねえ、コノハちゃんの弟子くん？\nわたしからあなたにおねがいがあるの。,Hey; Konoha's disciple, I have a favor to ask you.
+あなたが満足するまでわたしたちに挑戦したら、\nあなたはギルドに「盗賊はもう退治しました」って報告してほしいの❤\nこんなに気持ちよくしてあげてるだもん。それぐらいいいでしょ？,Keep challenging us until you are satisfied; In exchange I want you to report to the guild that you have killed the bandits. That's good enough for you; isn't it?
+その報告と一緒に、わたしたちはこの街を去るわ。\nあんまり盗り過ぎて村の生活が成り立たなくなるのは嫌だし、\n豊かなところを転々としながら襲うぐらいで行くつもりだから。,With that report; we will leave the city. We have been drawing too much attention and the next person they send might be even stronger. We will go and seek greener pastures to plunder.
```
This line has different nuance. They're leaving because she doesn't like taking too much from an individual village.  
 
> With that report; we'll leave this town. I don't like stealing so much that villages can't sustain themselves; so we plan to move around; just attacking prosperous places as we go.

ArdynXT replied:
I will change it. I didn't really know necro well from the zombie girl translation so I thought she would want to come off as hiding the real reason why she would leave the town

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ねえ、コノハちゃんの弟子くん？\nわたしからあなたにおねがいがあるの。,Hey; Konoha's disciple, I have a favor to ask you.
+あなたが満足するまでわたしたちに挑戦したら、\nあなたはギルドに「盗賊はもう退治しました」って報告してほしいの❤\nこんなに気持ちよくしてあげてるだもん。それぐらいいいでしょ？,Keep challenging us until you are satisfied; In exchange I want you to report to the guild that you have killed the bandits. That's good enough for you; isn't it?
+その報告と一緒に、わたしたちはこの街を去るわ。\nあんまり盗り過ぎて村の生活が成り立たなくなるのは嫌だし、\n豊かなところを転々としながら襲うぐらいで行くつもりだから。,With that report; we will leave the city. We have been drawing too much attention and the next person they send might be even stronger. We will go and seek greener pastures to plunder.
+もしギルドに嘘の報告をしてくれたら、\nあなたのことも一緒に連れて行ってあげる。\nこの村から出たあと落ち合う場所を教えてあげるわ。,If you report that lie to the guild; I will take you with me. I will tell you where to meet up with us after you leave this village. 
```
after **we** leave this village.

(I think she's telling him where the next base will be)

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あなたが満足するまでわたしたちに挑戦したら、\nあなたはギルドに「盗賊はもう退治しました」って報告してほしいの❤\nこんなに気持ちよくしてあげてるだもん。それぐらいいいでしょ？,Keep challenging us until you are satisfied; In exchange I want you to report to the guild that you have killed the bandits. That's good enough for you; isn't it?
+その報告と一緒に、わたしたちはこの街を去るわ。\nあんまり盗り過ぎて村の生活が成り立たなくなるのは嫌だし、\n豊かなところを転々としながら襲うぐらいで行くつもりだから。,With that report; we will leave the city. We have been drawing too much attention and the next person they send might be even stronger. We will go and seek greener pastures to plunder.
+もしギルドに嘘の報告をしてくれたら、\nあなたのことも一緒に連れて行ってあげる。\nこの村から出たあと落ち合う場所を教えてあげるわ。,If you report that lie to the guild; I will take you with me. I will tell you where to meet up with us after you leave this village. 
+想像して？用心棒生活はけっこう気楽で楽しいのよ。\nわたしたちは面倒ごとか起きるまでは、一日中寝転んでればいいの。,Imagine that? The life of a bandit is quite carefree and fun. We can lie around all day until something troublesome happens.
```
bandit -> bodyguard

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+想像して？用心棒生活はけっこう気楽で楽しいのよ。\nわたしたちは面倒ごとか起きるまでは、一日中寝転んでればいいの。,Imagine that? The life of a bandit is quite carefree and fun. We can lie around all day until something troublesome happens.
+ひまなときは研究でも修行でも好きにできるし、\n食べ物だってたくさん食べられる。それに・・・。,In my free time; I can do whatever I want; whether it be research; training; and eating a lot of food. Plus… 
+たまにはあなたにエッチなこともしてあげるわ。\n私の魔力補充もかねてだけど❤,I will do naughty things to you once in a while. You'll be our personal mana supply~❤
+とても楽しそうな生活でしょ？\nねえ、ギルドへの嘘の報告。おねがいね❤,Sounds like fun; doesn't it?❤
```
Missing last bit, after "Sounds like fun; doesn't it?" add one of these or mix and-match:
> Hey~ I need you to submit that false report to the guild… you'll do that for me; right?❤

or 
> Oh; and about that false report to the guild… you'll take care of it for me; won't you?❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+あっそう。それじゃあ・・・❤,Oh yeah? Well then... Zombie girl~❤
+了解、マスター。パイズリ早くするね・・・❤,Roger that; Master. This tit job will finish him quickly…❤
+うあ、あううっ❤❤きもちいいっ・・・・❤きもちいい・・・❤❤,Ohhhh; ahhhhh!❤❤❤ Yes; that feels good ・・・・❤ Yes; that feels good...❤❤
+その後、何度かネクロに挑戦を試みたが、すっかり彼女たちの快楽におぼれてしまった僕は、決して彼女たちに勝つことはできなかった。,After that; I tried several times to challenge necro; but I was never able to beat them; as I was completely drowned in their pleasure.
```
them -> her

ArdynXT replied:
I thought he was referring to zombie girl and necro since they both ambushed him in the defeat scene I will change it to her

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+うあ、あううっ❤❤きもちいいっ・・・・❤きもちいい・・・❤❤,Ohhhh; ahhhhh!❤❤❤ Yes; that feels good ・・・・❤ Yes; that feels good...❤❤
+その後、何度かネクロに挑戦を試みたが、すっかり彼女たちの快楽におぼれてしまった僕は、決して彼女たちに勝つことはできなかった。,After that; I tried several times to challenge necro; but I was never able to beat them; as I was completely drowned in their pleasure.
+それどころか開始と同時に{自分}の姿に見とれてしまい、カードを１枚も召喚することもできず、そのまま{自分}の胸や口でイかされてしまうこともある始末だった。,In fact; as soon as the duel started; I was so taken by the sight of {自分} that I could not summon a single card and was instantly forced to cum on her breasts or in her mouth.
+そして・・・。,And…
```
And **then...**

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+その後、何度かネクロに挑戦を試みたが、すっかり彼女たちの快楽におぼれてしまった僕は、決して彼女たちに勝つことはできなかった。,After that; I tried several times to challenge necro; but I was never able to beat them; as I was completely drowned in their pleasure.
+それどころか開始と同時に{自分}の姿に見とれてしまい、カードを１枚も召喚することもできず、そのまま{自分}の胸や口でイかされてしまうこともある始末だった。,In fact; as soon as the duel started; I was so taken by the sight of {自分} that I could not summon a single card and was instantly forced to cum on her breasts or in her mouth.
+そして・・・。,And…
+うふふ、新しい隠れ家もなかなかいいところね。\nこのあたりには富豪の屋敷があるというし、\nそこを襲撃していくことになるかしら。,Hmmm...The new hideout is quite nice. I heard that this is quite the wealthy town. We'll have to pay them a visit.
```
Hmmm...  -> Ufufu;

We'll have to pay them a visit. -> I wonder if we'll end up raiding them.

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+うふふ、新しい隠れ家もなかなかいいところね。\nこのあたりには富豪の屋敷があるというし、\nそこを襲撃していくことになるかしら。,Hmmm...The new hideout is quite nice. I heard that this is quite the wealthy town. We'll have to pay them a visit.
+ね、新しい生活はどう？\n一生懸命正しいことをしてきたあなたには、\nまだ慣れないかもしれないけれど・・・。,Hey; how is your new life? You have always worked so hard to do the right thing; you may not be used to this yet.…
+一緒に爛れた生活に溺れていきましょう？\n研究もできるし、おいしいご飯もあるし、いい生活よ。\nって、ふふ、その目・・・。早くわたしたちに甘えたいのね❤,Let's drown in the blissful life together; shall we? We can do research and have good food; I can see the lust in your eyes.... You can't wait to be pampered by us~❤
+くすくす・・・❤仲間になってもやることは変わらないね・・・。\nほら、おっぱいでしてあげる・・・❤\nあとで少しだけ、肩をかじらせてね・・・❤,*Giggle*... ❤You're still drooling over me....❤ I'll give you a titty fuck...❤ So let me nibble on your shoulder after...❤
```
> *Giggle&ast;... ❤You're still drooling over **me; even though you're one of us now...❤** I'll give you a titty fuck...❤ So let me nibble on your shoulder after...❤

WisDZ commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
+ね、新しい生活はどう？\n一生懸命正しいことをしてきたあなたには、\nまだ慣れないかもしれないけれど・・・。,Hey; how is your new life? You have always worked so hard to do the right thing; you may not be used to this yet.…
+一緒に爛れた生活に溺れていきましょう？\n研究もできるし、おいしいご飯もあるし、いい生活よ。\nって、ふふ、その目・・・。早くわたしたちに甘えたいのね❤,Let's drown in the blissful life together; shall we? We can do research and have good food; I can see the lust in your eyes.... You can't wait to be pampered by us~❤
+くすくす・・・❤仲間になってもやることは変わらないね・・・。\nほら、おっぱいでしてあげる・・・❤\nあとで少しだけ、肩をかじらせてね・・・❤,*Giggle*... ❤You're still drooling over me....❤ I'll give you a titty fuck...❤ So let me nibble on your shoulder after...❤
+その後、{主人公}がどこに行ったかは誰も知らない。無敗のルーキーの物語は、こうしてその幕を閉じたのだった。,No one knows where the hero {主人公} went after that. Thus ended the story of the undefeated rookie.
```
> No one knows where **~the hero~** {主人公} went after that. Thus ended the story of the undefeated rookie.

("Hero" came from the brackets "{主人公}", but since it's replaced by the player's name, it shouldn't be included)

WisDZ reviewed (commented):
Review is done! Apologies if I went overboard with the suggestions.
Feel free to reply here or in Discord if you prefer.

I'm still fairly busy with irl stuff through the end of the year, but I'm hoping to be quicker on other reviews regardless. No promises, but I'll try! ٩(˶ᵔ ᵕ ᵔ˶ )و

ArdynXT committed:
Second batch of issues

Fixed issues from the second batch will be going for the third batch soon!

Giratena reviewed (changes requested):
Marking as changes requested on Wiz's behalf ^^

Giratena committed:
Fix commas

ArdynXT committed:
All Wiz conflicts resolved/adjusted

Giratena committed:
Fix description

Giratena commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,139 +1,138 @@
-くすくす・・・❤仲間になってもやることは変わらないね・・・。\nほら、おっぱいでしてあげる・・・❤\nあとで少しだけ、肩をかじらせてね・・・❤,
-その後、{主人公}がどこに行ったかは誰も知らない。無敗のルーキーの物語は、こうしてその幕を閉じたのだった。,
+　ゾンビのモンスター娘。体はひんやりと冷たいが、胸やふとももは柔らかい。相手にゾンビウイルスを感染させるために口を使った誘惑をしてくるが、そうした本能とは別に、性的なことを通して「命」を強く感じられるため元々エッチなことが好き。,A zombie monster girl. Her skin is cold to the touch yet her breasts and thighs are soft and squishy. She uses her mouth to spread the infection of her zombie virus. A lust for carnal desires that makes her feel alive. 
+口についている赤いものは血ではなくザクロの果汁である。肉を生で食べるのは不衛生だと言ってネクロが赤い果物を代わりに与えたところ、本人は気に入ってよくザクロをかじっている。,An insatiable craving for human flesh; Necro displeased with her cravings gave her pomegranate instead; her reason being raw meat was unsanitary; The zombie girl took a liking to the red fruit and can be often seen eating them. In fact the red around her lips is not blood but actually pomegranate juice.
```
You changed thr Japanese lines here. Was this intentional? It causes the git changes to look funny but if it makes the translation work we can keep it.

ArdynXT replied:
I'll check my copy since I believe this is probably a typo.

ArdynXT replied:
Just a typo there is still a problem with getting the bio to work I have a hunch on how to fix it

ArdynXT replied:
I checked the chinese translation to see how they did it but I could not find it there. I am not sure if someone wrote those bios by hand but there is probably some type of typo there and in the bio or we just need to account for the spaces/line breaks I haven't had to do those before so I am not sure what kinda tricks we need to get them working.

Giratena replied:
Ok lets leave it as is for now.

Giratena committed:
Commas

Giratena commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -30,15 +30,15 @@ EU53_paizuriAnime,
 命が危なくなると、子孫を残すために、\nペニスが固くなるっていうけど、\nほんとみたい・・・❤,They say that when your life is in danger; your dick gets hard as a last resort to preserve your genes. It seems like that's true...❤
 噛まれる前よりガチガチに固くなって、\nシコシコするたびに顔がどんどん蕩けて、\nふふ・・・❤子孫を残すために、びゅってしたくてたまらないんだね❤,It's become much harder than before you were bitten; and every time you stroke it; that naughty expression on your face becomes even harder to hide; fufu...❤ To ensure you leave offspring; you can't help wanting to cum; right?❤
-ほら、あなたの子種汁、わたしのスカートにかけていいよ・・・❤\n女の子の「子供を作るところ」に、射精して、\nそれで自分の命を終わらせちゃおう・・・？,Here; you can pour your seed juice underneath my skirt...❤ Will you release on my "child making place" and end your life along with it...?
+ほら、あなたの子種汁、わたしのスカートにかけていいよ・・・❤\n女の子の「子供を作るところ」に、射精して、\nそれで自分の命を終わらせちゃおう・・・？,"Here; you can pour your seed juice underneath my skirt...❤ Will you release on my ""child making place"" and end your life along with it...?"
```
This sentence here is quite unusual, more so than what this character would normally say.

Maybe something like this, (feel free to revise):
"Come; spill your seed beneath my skirt...❤ Ejaculate on my "baby making place" and end your life along with it..."

ArdynXT replied:
This line is very strange.... 
"Come; spill your seed beneath my skirt...❤ Ejaculate on my "baby making place" as I take your life along with it..."
Do you feel like it's redundant to tell him to cum twice? 
Maybe we either stick to skirt or babymaking place?

Giratena replied:
The second sentence is an extension of the previous so I think it's OK.

Giratena commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -67,72 +67,72 @@ EU53_paizuriAnime,
 くすくす…❤わたしのここに精液かけられて、満足した・・・？\nオナニーで負けちゃうなんて、へんたいだね・・・❤,*Giggle*...❤ Are you satisfied with your seed being spilled here? I didn’t even have to touch you...❤
 乳首が見れなくて残念だったね…❤\nでも、わたしのおっぱい気持ちよかったなら、\nそれで十分でしょ。くすくす…❤,Too bad you couldn't see my nipples...❤ But if my tits felt that good; then I’m sure you can’t complain *Giggle*❤
-んっ・・・❤見て、こんなにたくさん出しちゃったね・・・❤\n生気にあふれてて、おいしい・・・❤ごくん・・・❤,Look how much you’ve cum...❤ Lush and delicious...❤ Gasp...❤
+んっ・・・❤見て、こんなにたくさん出しちゃったね・・・❤\n生気にあふれてて、おいしい・・・❤ごくん・・・❤,Look how much you’ve cum...❤ So delicious and full of life...❤ I'll savor every drop...❤
```
Good change!

ArdynXT replied:
Shrug always coming in clutch

Giratena commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -67,72 +67,72 @@ EU53_paizuriAnime,
-ダメよ。わたし殺しはやらない主義なの。分かってるでしょ。,No; you can't eat him! I'm a not a killer! But you may pleasure him to restore my mana.
-ふふ、マスターのやさしさに感謝してね・・・。\nでも死ぬ少し手前まで、たっぷり搾り取ってあげる・・・❤,Thank you for your kindness Master.... I'll keep squeezing you for your precious mana...❤
+ダメよ。わたし殺しはやらない主義なの。分かってるでしょ。,Absolutely not! You know the rules; we do not kill; But you may pleasure him to restore my mana.
+ふふ、マスターのやさしさに感謝してね・・・。\nでも死ぬ少し手前まで、たっぷり搾り取ってあげる・・・❤,Fufu; be grateful for Master's kindness… But I'll still thoroughly drain you until you're just short of dying…❤
```
Use ... instead of …

Giratena commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -67,72 +67,72 @@ EU53_paizuriAnime,
+ふふふ、かわいいじゃない❤\n男の子なんてそれぐらいでいいのよ❤,Fufufu; aren't you just adorable~❤ This is exactly what boys should be like.❤
 ほら、わたしも{自分}のお手伝い。\n顔におっぱいを乗せてあげる❤,Here; I'll help out {自分}; by putting my boobs on your face❤
-盗賊の用心棒のおっぱいなんかに負けちゃダメよ。\nコノハちゃんの弟子くん❤\nえい❤,Look at you losing yourself in the embrace of a bandits chest.
+盗賊の用心棒のおっぱいなんかに負けちゃダメよ。\nコノハちゃんの弟子くん❤\nえい❤,Don't lose to the breasts of a mere bodyguard; Konoha's little apprentice❤
```
Full stop after apprentice

ArdynXT replied:
Keep the heart though right? I added a period here cause I assume she is teasing him here but I felt like the heart fits thoughts?

Giratena replied:
Yes full stop after apprentice, obviously keep the heart ^^

Giratena commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -67,72 +67,72 @@ EU53_paizuriAnime,
+ふふ❤わたしのおっぱいやわらかいでしょう？\n{自分}のと比べるとあったかいから、\n余計にふわふわに感じちゃうかも❤,Fufu~❤ My boobs are soft; aren't they? They are much warmer than my {自分}; so they might feel even fluffier...❤
+おっぱいに埋もれて表情は見えないけど・・・❤\n腕も脚も完全に力が抜けてる・・・❤\nふふ、きっと幸せそうな顔になっているんでしょうね❤,I can't see his expression because he’s buried in your boobs... ❤❤ his arms and legs are completely relaxed...❤ Fufu; he must be grinning ear to ear.❤
+おちんちんも目で見て分かるぐらい固くなってるわ❤\n{自分}の胸の中からこぼれそうなぐらい、びくびく跳ねてる❤,Your penis has gotten so hard I can see it clearly.❤ It's twitching and jumping so much; it looks like it might spill out from {自分}'s cleavage.❤
+くす・・・❤捕まえた獲物は逃がさない・・・❤\nそんなにおちんちん暴れさせても、\nこのおっぱいからは逃げられないよ・・・❤,Hehe…❤ There is no escape once I've captured my prey...❤ No matter how much you helplessly thrash and thrust your cock about; you won't escape from these breasts…❤
```
Use ... after hehe and breasts

Giratena commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -67,72 +67,72 @@ EU53_paizuriAnime,
+おちんちんも目で見て分かるぐらい固くなってるわ❤\n{自分}の胸の中からこぼれそうなぐらい、びくびく跳ねてる❤,Your penis has gotten so hard I can see it clearly.❤ It's twitching and jumping so much; it looks like it might spill out from {自分}'s cleavage.❤
+くす・・・❤捕まえた獲物は逃がさない・・・❤\nそんなにおちんちん暴れさせても、\nこのおっぱいからは逃げられないよ・・・❤,Hehe…❤ There is no escape once I've captured my prey...❤ No matter how much you helplessly thrash and thrust your cock about; you won't escape from these breasts…❤
+ほらほらほら・・・・❤\nおっぱいでもぐもぐされて、おちんちんの中身全部食べられちゃえ・・・❤,Come on; come on; that's it...❤ Get devoured by my breasts; and let them drain everything out of your cock...❤
+まだまだ出るでしょ？ふふ・・・❤\n生命エネルギー、どんどんちょうだい・・・❤,You can still cum more; can't you? Fufu…❤ Give me more and more of your life energy…❤
```
... here as well

Giratena commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -67,72 +67,72 @@ EU53_paizuriAnime,
 それどころか開始と同時に{自分}の姿に見とれてしまい、カードを１枚も召喚することもできず、そのまま{自分}の胸や口でイかされてしまうこともある始末だった。,In fact; as soon as the duel started; I was so taken by the sight of {自分} that I could not summon a single card and was instantly forced to cum on her breasts or in her mouth.
-そして・・・。,And…
-うふふ、新しい隠れ家もなかなかいいところね。\nこのあたりには富豪の屋敷があるというし、\nそこを襲撃していくことになるかしら。,Hmmm...The new hideout is quite nice. I heard that this is quite the wealthy town. We'll have to pay them a visit.
+そして・・・。,And then…
```
Later that day...

Giratena reviewed (changes requested):
Looking better! Just some more changes/comments @ArdynXT

ArdynXT committed:
Addressing Issues 2

ArdynXT committed:
Grammatical fixes

Removed spaces and triple ... that were copy and pasted rather than typed

Giratena commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -124,15 +124,15 @@ EU53_paizuriAnime,
 あっそう。それじゃあ・・・❤,Oh yeah? Well then... Zombie girl~❤
-了解、マスター。パイズリ早くするね・・・❤,Roger that; Master. This tit job will finish him quickly…❤
-うあ、あううっ❤❤きもちいいっ・・・・❤きもちいい・・・❤❤,Ohhhh; ahhhhh!❤❤❤ Yes; that feels good ・・・・❤ Yes; that feels good...❤❤
+了解、マスター。パイズリ早くするね・・・❤,Roger that Master; This tit job will finish him quickly...❤
```
Master; -> Master.

Giratena commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -38,8 +38,8 @@ EU53_paizuriAnime,
-ほら、あなたの子種汁、わたしのスカートにかけていいよ・・・❤\n女の子の「子供を作るところ」に、射精して、\nそれで自分の命を終わらせちゃおう・・・？,"Here; you can pour your seed juice underneath my skirt...❤ Will you release on my ""child making place"" and end your life along with it...?"
-もちろんわたしにかけても子供なんてできないけど・・・❤\nでもそれでもいいよね・・・❤しこしこ気持ちいいもんね・・・❤\nくすくす❤,Of course it’s all for nothing; I can’t have kids...❤  *Giggle* ~❤ It feels so good to rub it in...❤
+ほら、あなたの子種汁、わたしのスカートにかけていいよ・・・❤\n女の子の「子供を作るところ」に、射精して、\nそれで自分の命を終わらせちゃおう・・・？,Come; spill your seed beneath my skirt...❤ Ejaculate on my "baby making place" as I take your life along with it...
+もちろんわたしにかけても子供なんてできないけど・・・❤\nでもそれでもいいよね・・・❤しこしこ気持ちいいもんね・・・❤\nくすくす❤,Of course it’s all for nothing; I can’t have kids...❤  *Giggle* ~❤ But it feels so good to stroke your cock; doesn't it...
```
Space before ~

Giratena reviewed (changes requested):
Just two more comments!

Giratena committed:
Revise Zombie Girl card text and translations

Updated translations and descriptions for the Zombie Girl card, including adjustments to the English text for clarity and consistency.
